# ncbi_scripts
simple scripts for querying pubmed and clinvar via various inputs

## Usage
`head` each script for clarification on usage

```
$ head gene_abstracts.sh
### Usage
# bash gene_abstracts.sh $GENE
# where $GENE is a HUGO style gene name like GFI1, TP53, BRCA1, DDX17, etc
# Example: bash gene_abstracts.sh BRCA1
```

## Example
get titles and abstracts for the gene GFI1

```
$ bash gene_abstracts.sh GFI1
>>> GFI1

Title: Chromosomal localization of a gene, GF1, encoding a novel zinc finger protein reveals a new syntenic region between man and rodents.
Abstract: The Gfi1 gene encodes a zinc finger protein which binds DNA and is involved in transcriptional regulation. Gfi1 was assigned to the central portion of mouse Chr 5 by interspecific backcross mapping and to human chromosome band 1p22 and rat chromosome band 14p22 by fluorescence in situ hybridization (FISH). Comparative mapping data presented here describes a new syntenic region between man and rodents.

Title: Progression of interleukin-2 (IL-2)-dependent rat T cell lymphoma lines to IL-2-independent growth following activation of a gene (Gfi-1) encoding a novel zinc finger protein.
Abstract: During progression of Moloney murine leukemia virus (Mo-MuLV)-induced rat T cell lymphomas, growth selection results in the expansion of cell clones carrying increasing numbers of integrated proviruses. These new provirus insertions reproducibly contribute to enhanced growth, allowing the emergence of cell clones from the initially heterogeneous population of tumor cells. The Mo-MuLV-induced rat T cell lymphoma lines 2780d and 5675d, which are dependent on interleukin-2 (IL-2) for growth in culture (IL-2d), were placed in IL-2-free medium to select for IL-2-independent (IL-2i) mutants. Southern blot analysis of genomic DNA from these mutants, which was hybridized to a Mo-MuLV long terminal repeat probe, revealed that all mutants carried new provirus insertions (from one to four new proviruses per cell line). A locus of integration identified through cloning of the single new provirus detected in one of the IL-2i mutants, 2780i.5, was found to be the target of provirus insertion in 1 additional IL-2i cell line of 24 tested. A full-length cDNA of a gene (growth factor independence-1 [Gfi-1]) activated by promoter insertion in the 2780i.5 cells was cloned and shown to encode a novel zinc finger protein. Gfi-1 is expressed at low levels in IL-2d cell lines cultured in IL-2-containing medium and at high levels in most IL-2i cell lines, including the two harboring a provirus at this locus. Gfi-1 expression in adult animals is restricted to the thymus, spleen, and testis. In mitogen-stimulated splenocytes, Gfi-1 expression begins to rise at 12 h after stimulation and reaches very high levels after 50 h, suggesting that it may be functionally involved in events occurring after the interaction of IL-2 with its receptor, perhaps during the transition from the G1 to the S phase of the cell cycle. In agreement with this, Gfi-1 does not induce the expression of IL-2. Expression of Gfi-1 in 2780d cells following transfer of a Gfi-1/LXSN retrovirus construct contributes to the emergence of the IL-2i phenotype.

Title: Gfi-1 encodes a nuclear zinc finger protein that binds DNA and functions as a transcriptional repressor.
Abstract: The Gfi-1 proto-oncogene encodes a zinc finger protein with six C2H2-type, C-terminal zinc finger motifs and is activated by provirus integration in T-cell lymphoma lines selected for interleukin-2 independence in culture and in primary retrovirus-induced thymomas. Gfi-1 expression in adult animals is restricted to the thymus, spleen, and testis and is enhanced in mitogen-stimulated splenocytes. In this report, we show that Gfi-1 is a 55-kDa nuclear protein that binds DNA in a sequence-specific manner. The Gfi-1 binding site, TAAATCAC(A/T)GCA, was defined via random oligonucleotide selection utilizing a bacterially expressed glutathione S-transferase-Gfi-1 fusion protein. Binding to this site was confirmed by electrophoretic mobility shift assays and DNase I footprinting. Methylation interference analysis and electrophoretic mobility shift assays with mutant oliginucleotides defined the relative importance of specific bases at the consensus binding site. Deletion of individual zinc fingers demonstrated that only zinc fingers 3, 4, and 5 are required for sequence-specific DNA binding. Potential Gfi-1 binding sites were detected in a large number of eukaryotic promoter-enhancers, including the enhancers of several proto-oncogenes and cytokine genes and the enhancer of the human cytomegalovirus (HCMV) major immediate-early promoter, which contains two such sites. HCMV major immediate-early-chloramphenicol acetyltransferase reporter constructs, transfected into NIH 3T3 fibroblasts, were repressed by Gfi-1, and the repression was abrogated by mutation of critical residues in the two Gfi-1 binding sites. These results suggest that Gfi-1 may play a role in HCMV biology and may contribute to oncogenesis and T-cell activation by repressing the expression of genes that inhibit these processes.

Title: Cloning of the human Gfi-1 gene and its mapping to chromosome region 1p22.
Abstract: Recently the rat and mouse Growth Factor Independence (Gfi-1) genes have been cloned (Gilks et al., 1993; Zoring et al; 1996). This gene allows cells in culture to overcome the depletion of growth factors in the culture medium and maintain their proliferative potential. As part of a cloning strategy to isolated genes from human chromosome 1p22 which are associated with a constitutional chromosome translocation from a patient with stage 4S neuroblastoma, we have identified the human homologue of the Gfi gene and defined a 50 Kb map position within a well characterised YAC contig from the region. The full length cDNA sequence is 81% homologous with the rodent counterparts and, at the protein level, is even more highly conserved.

Title: Shotgun sequencing of the human transcriptome with ORF expressed sequence tags.
Abstract: Theoretical considerations predict that amplification of expressed gene transcripts by reverse transcription-PCR using arbitrarily chosen primers will result in the preferential amplification of the central portion of the transcript. Systematic, high-throughput sequencing of such products would result in an expressed sequence tag (EST) database consisting of central, generally coding regions of expressed genes. Such a database would add significant value to existing public EST databases, which consist mostly of sequences derived from the extremities of cDNAs, and facilitate the construction of contigs of transcript sequences. We tested our predictions, creating a database of 10,000 sequences from human breast tumors. The data confirmed the central distribution of the sequences, the significant normalization of the sequence population, the frequent extension of contigs composed of existing human ESTs, and the identification of a series of potentially important homologues of known genes. This approach should make a significant contribution to the early identification of important human genes, the deciphering of the draft human genome sequence currently being compiled, and the shotgun sequencing of the human transcriptome.

Title: The zinc finger protein Gfi-1 can enhance STAT3 signaling by interacting with the STAT3 inhibitor PIAS3.
Abstract: STAT factors act as signal transducers of cytokine receptors and transcriptionally activate specific target genes. The recently discovered protein PIAS3 binds directly to STAT3 and blocks transcriptional activation. Here, we present experimental evidence implementing the zinc finger protein Gfi-1 as a new regulatory factor in STAT3-mediated signal transduction. The interaction between the two proteins first became evident in a yeast two-hybrid screen but was also seen in coprecipitation experiments from eukaryotic cells. Moreover, we found that both Gfi-1 and PIAS3 colocalize in a characteristic nuclear dot structure. While PIAS3 exerts a profound inhibitory effect on STAT3-mediated transcription of target promoters, Gfi-1 can overcome the PIAS3 block and significantly enhances STAT3-mediated transcriptional activation. In primary T cells, Gfi-1 was able to amplify IL-6-dependent T-cell activation. As Gfi-1 is a known, dominant proto-oncogene, our findings bear particular importance for the recently described ability of STAT3 to transform cells malignantly and offer an explanation of the oncogenic potential of Gfi-1 in T lymphocytes.

Title: Erythroid expansion mediated by the Gfi-1B zinc finger protein: role in normal hematopoiesis.
Abstract: In the search for genes expressed in hematopoietic stem cells, we identified that the expression of Gfi-1B (growth factor independence-1B) is highly restricted to hematopoietic stem cells, erythroblasts, and megakaryocytes. Gfi-1 and Gfi-1B are zinc finger proteins that share highly conserved SNAG and 6 zinc finger domains. Gfi-1 has been characterized as an oncogene involved in lymphoid malignancies in mice. In contrast, role of Gfi-1B in hematopoiesis has not been well characterized. In this study, we analyzed its function in human hematopoiesis. Enforced expression of Gfi-1B in human CD34(+) hematopoietic progenitors induced a drastic expansion of erythroblasts in an erythropoietin-independent manner. Expression of Gfi-1B did not promote erythroid commitment, but enhanced proliferation of immature erythroblasts. Erythroblasts expanded by exogenous Gfi-1B, however, failed to differentiate beyond proerythroblast stage and showed massive apoptosis. These biologic effects of Gfi-1B were mediated through its zinc finger domain, but not by the SNAG or non-zinc finger domain. Proliferation of erythroblasts was associated with sustained expression of GATA-2 but not of GATA-1, indicating a potential link between Gfi-1B and GATA family regulators. Importantly, the function of Gfi-1B to modulate transcription was dependent on promoter context. In addition, activation of transcription of an artificial promoter was mediated through its zinc finger domain. These findings establish Gfi-1B as a novel erythroid regulator and reveal its specific involvement in the regulation of erythroid cell growth through modulating erythroid-specific gene expression.

Title: Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.
Abstract: The National Institutes of Health Mammalian Gene Collection (MGC) Program is a multiinstitutional effort to identify and sequence a cDNA clone containing a complete ORF for each human and mouse gene. ESTs were generated from libraries enriched for full-length cDNAs and analyzed to identify candidate full-ORF clones, which then were sequenced to high accuracy. The MGC has currently sequenced and verified the full ORF for a nonredundant set of &gt;9,000 human and &gt;6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and 3,500 mouse genes also have been identified. All MGC sequences and clones are available without restriction through public databases and clone distribution networks (see http:mgc.nci.nih.gov).

Title: Mutations in proto-oncogene GFI1 cause human neutropenia and target ELA2.
Abstract: Mice lacking the transcriptional repressor oncoprotein Gfi1 are unexpectedly neutropenic. We therefore screened GFI1 as a candidate for association with neutropenia in affected individuals without mutations in ELA2 (encoding neutrophil elastase), the most common cause of severe congenital neutropenia (SCN; ref. 3). We found dominant negative zinc finger mutations that disable transcriptional repressor activity. The phenotype also includes immunodeficient lymphocytes and production of a circulating population of myeloid cells that appear immature. We show by chromatin immunoprecipitation, gel shift, reporter assays and elevated expression of ELA2 in vivo in neutropenic individuals that GFI1 represses ELA2, linking these two genes in a common pathway involved in myeloid differentiation.

Title: Gfi-1 attaches to the nuclear matrix, associates with ETO (MTG8) and histone deacetylase proteins, and represses transcription using a TSA-sensitive mechanism.
Abstract: Gfi-1 and Gfi-1B can repress transcription and play important roles in hematopoietic cell survival and differentiation. Although these proteins are known to bind DNA through a C-terminal zinc-finger domain and may require an N-terminal SNAG domain (SNAIL/Gfi-1) to repress transcription, the mechanism by which Gfi-1 and Gfi-1B act is unknown. A first step towards understanding the mechanism by which these proteins repress transcription is to identify interacting proteins that could contribute to transcriptional repression. ETO (also termed MTG8), was first identified through its involvement in the (8;21) translocation associated with acute myelogenous leukemia. It attaches to the nuclear matrix and associates with histone deacetylases and the co-repressors N-CoR, SMRT, and mSin3A, and may act as a co-repressor for site-specific transcriptions factors. In this report we demonstrate that Gfi-1 interacts with ETO and related proteins both in vitro and in vivo and with histone deacetylase proteins in vivo. We observed that a portion of Gfi-1 and Gfi-1B associated with the nuclear matrix, as is the case with ETO. Moreover, Gfi-1 and ETO co-localize to punctate subnuclear structures. When co-expressed in mammalian cells, Gfi-1 associates with histone deacetylse-1 (HDAC-1), HDAC-2, and HDAC-3. These data identify ETO as a partner for Gfi-1 and Gfi-1B, and suggest that Gfi-1 proteins repress transcription through recruitment of histone deacetylase-containing complexes.

Title: Complete sequencing and characterization of 21,243 full-length human cDNAs.
Abstract: As a base for human transcriptome and functional genomics, we created the "full-length long Japan" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.

Title: Targeted transcriptional repression of Gfi1 by GFI1 and GFI1B in lymphoid cells.
Abstract: Growth factor independence-1 (GFI1) and GFI1B are closely related, yet differentially expressed transcriptional repressors with nearly identical DNA binding domains. GFI1 is upregulated in the earliest thymocyte precursors, while GFI1B expression is restricted to T lymphopoiesis stages coincident with activation. Transgenic expression of GFI1 potentiates T-cell activation, while forced GFI1B expression decreases activation. Both mice and humans with mutant Gfi1 display lymphoid abnormalities. Here we describe autoregulation of Gfi1 in primary mouse thymocytes and a human T-cell line. GFI1 binding to cis-element sequences conserved between rat, mouse and human Gfi1 mediates direct and potent transcriptional repression. In addition, dramatic regulation of Gfi1 can also be mediated by GFI1B. These data provide the first example of a gene directly targeted by GFI1 and GFI1B. Moreover, they support a role for auto- and trans-regulation of Gfi1 by GFI1 and GFI1B in maintaining the normal expression patterns of Gfi1, and suggest that GFI1B may indirectly affect T-cell activation through repression of Gfi1.

Title: The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).
Abstract: The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.

Title: Identification of growth factor independent-1 (GFI1) as a repressor of 25-hydroxyvitamin D 1-alpha hydroxylase (CYP27B1) gene expression in human prostate cancer cells.
Abstract: The hormone 1,25-dihydroxyvitamin D (1,25D) may play a protective role in prostate cancer. 25-hydroxyvitamin D 1-alpha hydroxylase (CYP27B1) is the enzyme responsible for the regulation of cellular 1,25D levels. CYP27B1 is substantially repressed in prostate cancer cells. We have investigated the molecular basis for this inhibition. First, we identify a repressive region between -997 and -1200 in the human CYP27B1 promoter following transient transfection analysis in the prostate cancer cell lines DU145, PC3 and LNCaP. Next, we demonstrate a role for the transcription factor growth factor independent-1 (GFI1) in the repression of CYP27B1. Electrophoretic mobility assays with nuclear extracts from prostate cancer cell lines established binding of GFI1 to the sequence 5'-TGGTACAATCATAACTCACTGCAG-3' present at -997 to -1200 in the repressive region. Site directed mutagenesis of the core GFI1 binding sequence (5'-AATC-3') substantially increased while forced expression of GFI1 decreased the expression of the CYP27B1 reporter construct. Importantly, GFI1 repression is dependent on an intact GFI1 binding site in the -997 to -1200 region. GFI1 is an oncoprotein known to form a large protein complex with co-repressors that recruit histone deacetylases. We propose that the formation of such a repressive complex on the inhibitory domain of the CYP27B1 gene in prostate cancer cells could lead to silencing of either the nearby enhancer or proximal promoter domains and lead to cancer progression by reducing local production of 1,25D. These studies provide the basis for a more detailed understanding of CYP27B1 repression in prostate cancer cells and could provide a novel insight in future diagnosis and treatment.

Title: Gfi1 coordinates epigenetic repression of p21Cip/WAF1 by recruitment of histone lysine methyltransferase G9a and histone deacetylase 1.
Abstract: The growth factor independent 1 (Gfi1) transcriptional regulator oncoprotein plays a crucial role in hematopoietic, inner ear, and pulmonary neuroendocrine cell development and governs cell processes as diverse as self-renewal of hematopoietic stem cells, proliferation, apoptosis, differentiation, cell fate specification, and oncogenesis. However, the molecular basis of its transcriptional functions has remained elusive. Here we show that Gfi1 recruits the histone lysine methyltransferase G9a and the histone deacetylase 1 (HDAC1) in order to modify the chromatin of genes targeted for repression by Gfi1. G9a and HDAC1 are both in a repressive complex assembled by Gfi1. Endogenous Gfi1 colocalizes with G9a, HDAC1, and K9-dimethylated histone H3. Gfi1 associates with G9a and HDAC1 on the promoter of the cell cycle regulator p21Cip/WAF1, resulting in an increase in K9 dimethylation at histone H3. Silencing of Gfi1 expression in myeloid cells reverses G9a and HDAC1 recruitment to p21Cip/WAF1 and elevates its expression. These findings highlight the role of epigenetics in the regulation of development and oncogenesis by Gfi1.

Title: Zinc-finger transcription factor Gfi-1: versatile regulator of lymphocytes, neutrophils and hematopoietic stem cells.
Abstract: Gfi-1, originally identified as a transcriptional repressor in lymphoid cells, is now recognized to have essential, independent functions in neutrophil maturation and hematopoietic stem cell biology. Here we review recent studies pertaining to Gfi-1 and its cell context specific functions in hematopoietic development.

Title: Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes.
Abstract: By analyzing 1,780,295 5'-end sequences of human full-length cDNAs derived from 164 kinds of oligo-cap cDNA libraries, we identified 269,774 independent positions of transcriptional start sites (TSSs) for 14,628 human RefSeq genes. These TSSs were clustered into 30,964 clusters that were separated from each other by more than 500 bp and thus are very likely to constitute mutually distinct alternative promoters. To our surprise, at least 7674 (52%) human RefSeq genes were subject to regulation by putative alternative promoters (PAPs). On average, there were 3.1 PAPs per gene, with the composition of one CpG-island-containing promoter per 2.6 CpG-less promoters. In 17% of the PAP-containing loci, tissue-specific use of the PAPs was observed. The richest tissue sources of the tissue-specific PAPs were testis and brain. It was also intriguing that the PAP-containing promoters were enriched in the genes encoding signal transduction-related proteins and were rarer in the genes encoding extracellular proteins, possibly reflecting the varied functional requirement for and the restricted expression of those categories of genes, respectively. The patterns of the first exons were highly diverse as well. On average, there were 7.7 different splicing types of first exons per locus partly produced by the PAPs, suggesting that a wide variety of transcripts can be achieved by this mechanism. Our findings suggest that use of alternate promoters and consequent alternative use of first exons should play a pivotal role in generating the complexity required for the highly elaborated molecular systems in humans.

Title: A protein-protein interaction network for human inherited ataxias and disorders of Purkinje cell degeneration.
Abstract: Many human inherited neurodegenerative disorders are characterized by loss of balance due to cerebellar Purkinje cell (PC) degeneration. Although the disease-causing mutations have been identified for a number of these disorders, the normal functions of the proteins involved remain, in many cases, unknown. To gain insight into the function of proteins involved in PC degeneration, we developed an interaction network for 54 proteins involved in 23 inherited ataxias and expanded the network by incorporating literature-curated and evolutionarily conserved interactions. We identified 770 mostly novel protein-protein interactions using a stringent yeast two-hybrid screen; of 75 pairs tested, 83% of the interactions were verified in mammalian cells. Many ataxia-causing proteins share interacting partners, a subset of which have been found to modify neurodegeneration in animal models. This interactome thus provides a tool for understanding pathogenic mechanisms common for this class of neurodegenerative disorders and for identifying candidate genes for inherited ataxias.

Title: The growth factor independence-1 transcription factor: new functions and new insights.
Abstract: The growth factor independence-1 (Gfi1) transcription factor is required for proper development of neuroendocrine cells, sensory neurons, and blood. Patients with mutations in Gfi1 exhibit severe congenital neutropenia (SCN) or non-immune chronic idiopathic neutropenia of adults. Gfi1 was initially described as an oncoprotein that mediates tumor progression in a mouse model of leukemia; however, recent data suggest that Gfi1 may act as either an oncogene or an anti-proliferative tumor suppressor gene depending on the cell type. Here we review the latest literature on Gfi1, and emphasize its role in the hematopoietic, sensory and neuroendocrine systems.

Title: Diminished proteasomal degradation results in accumulation of Gfi1 protein in monocytes.
Abstract: Gfi1 is a transcriptional repressor essential during myeloid differentiation. Gfi1-/- mice exhibit a block in myeloid differentiation resulting in the accumulation of an immature myelo-monocytic cell population and the complete absence of mature neutrophils. Even though mRNA levels of Gfi1 appear to be very low in monocytes, Gfi1 might play a role in the monocytic lineage as Gfi1-/- mice exhibit diminished monocyte-derived dendritic cells and disturbed cytokine production by macrophages in response to LPS. We show here that Gfi1 protein levels are mainly regulated by the ubiquitin-proteasome system. Upon forced monocytic differentiation of U937 cells, Gfi1 mRNA levels dropped but protein levels increased due to diminished proteasomal turnover. Similarly, Gfi1 mRNA levels are low in primary monocytes whereas the protein is clearly detectable. Conversely, Gfi1 mRNA levels are high in granulocytes but the protein is swiftly degraded by the proteasome in these cells. Chromatin immunoprecipitation experiments showed that Gfi1 binds to the promoter of several granulocyte-specific genes in primary monocytes, including C/EBPalpha, neutrophil elastase, and Gfi1 itself. The binding of the repressor Gfi1 to these promoters correlated with low expression of these genes in monocytes compared with granulocytes. Our data fit a model in which Gfi1 protein levels are induced in primary monocytes, due to diminished proteasomal degradation, to repress genes that play a role in granulocytic differentiation.

Title: The transcriptional repressor GFI-1 antagonizes PU.1 activity through protein-protein interaction.
Abstract: Mice lacking the zinc finger transcriptional repressor protein GFI-1 are neutropenic. These mice generate abnormal immature myeloid cells exhibiting characteristics of both macrophages and granulocytes. Furthermore, Gfi-1(-/-) mice are highly susceptible to bacterial infection. Interestingly, Gfi-1(-/-) myeloid cells overexpress target genes of the PU.1 transcription factor such as the macrophage colony-stimulating factor receptor and PU.1 itself. We therefore determined whether GFI-1 modulates the transcriptional activity of PU.1. Our data demonstrate that GFI-1 physically interacts with PU.1, repressing PU.1-dependent transcription. This repression is functionally significant, as GFI-1 blocked PU.1-induced macrophage differentiation of a multipotential hematopoietic progenitor cell line. Retroviral expression of GFI-1 in primary murine hematopoietic progenitors increased granulocyte differentiation at the expense of macrophage differentiation. We interbred Gfi-1(+/-) and PU.1(+/-) mice and observed that heterozygosity at the PU.1 locus partially rescued the Gfi-1(-/-) mixed myeloid lineage phenotype, but failed to restore granulocyte differentiation. Our data demonstrate that GFI-1 represses PU.1 activity and that lack of this repression in Gfi-1(-/-) myeloid cells contributes to the observed mixed lineage phenotype.

Title: Role of oncoprotein growth factor independent-1 (GFI1) in repression of 25-hydroxyvitamin D 1alpha-hydroxylase (CYP27B1): a comparative analysis in human prostate cancer and kidney cells.
Abstract: 1,25-Dihydroxyvitamin D (1,25D) inhibits growth of prostate cancer cells and has been proposed to play a protective role in prostate cancer. However, 25-hydroxyvitamin D 1alpha-hydroxylase (CYP27B1), the enzyme responsible for the cellular synthesis of 1,25D, is repressed in prostate cancer cells. Recently, we have identified a role for the transcription factor, Growth Factor Independent-1 (GFI1) in the repression of the CYP27B1 gene in human prostate cancer cell lines. GFI1 is known to form a large protein complex with co-repressors that recruit histone deacetylases. We have proposed a model for the molecular repression of CYP27B1 gene expression. The formation of such a repressive complex on the inhibitory domain of the CYP27B1 gene in prostate cancer cells could lead to the silencing of gene expression either by inactivating nearby enhancer or proximal promoter domains and lead to cancer progression by reducing local production of 1,25D. These studies demonstrate that GFI1 may play a significant role in the down regulation of endogenous production of 1,25D in prostate cancer cells and could provide a novel insight to future diagnosis and treatment.

Title: Cooperative interaction between ETS1 and GFI1 transcription factors in the repression of Bax gene expression.
Abstract: The proto-oncoproteins ETS1 and growth factor independent-1 (GFI1) are implicated in cell growth and differentiation in various types of cells, and their deregulated expression is involved in malignant transformation. Here, we report that ETS1 and GFI1 interact and affect gene expression through their cross-talk. Co-immunoprecipitation analyses and glutathione-S-transferase pull-down assays revealed that ETS1 bound directly to GFI1 via its Ets domain, and GFI1 bound to ETS1 via its zinc-finger domain. Luciferase (Luc) assays using artificial reporters showed that GFI1 repressed ETS1-mediated transcriptional activation and ETS1 repressed GFI1-mediated transcriptional activation, in a dose-dependent manner. However, in the Bax promoter where the Ets- and Gfi-binding sites (EBS and GBS) are adjacent, ETS1 and GFI1 cooperatively reduced activation. Site-directed mutagenesis on the EBS and GBS of the Bax promoter showed that both binding sites were necessary for full repression. Chromatin immunoprecipitation analyses confirmed that an ETS1-GFI1 complex formed on the Bax promoter even when either EBS or GBS was mutated. Introduction of small interfering RNA against ETS1 and/or GFI1 enhanced endogenous Bax gene expression. Our results suggest that the interaction between ETS1 and GFI1 facilitates their binding to specific sites on the Bax promoter and represses Bax expression in vivo.

Title: Growth factor independence-1 (Gfi-1) plays a role in mediating specific granule deficiency (SGD) in a patient lacking a gene-inactivating mutation in the C/EBPepsilon gene.
Abstract: Neutrophil-specific granule deficiency (SGD) is a rare congenital disorder marked by recurrent bacterial infections. Neutrophils from SGD patients lack secondary and tertiary granules and their content proteins and lack normal neutrophil functions. Gene-inactivating mutations in the C/EBPepsilon gene have been identified in 2 SGD patients. Our studies on a third SGD patient revealed a heterozygous mutation in the C/EBPepsilon gene. However, we demonstrate elevated levels of C/EBPepsilon and PU.1 proteins in the patient's peripheral blood neutrophils. The expression of the transcription factor growth factor independence-1 (Gfi-1), however, was found to be markedly reduced in our SGD patient despite the absence of an obvious mutation in this gene. This may explain the elevated levels of both C/EBPepsilon and PU.1, which are targets of Gfi-1 transcriptional repression. We have generated a growth factor-dependent EML cell line from the bone marrow of Gfi-1(+/-) and Gfi-1(+/+) mice as a model for Gfi-1-deficient SGD, and demonstrate that lower levels of Gfi-1 expression in the Gfi-1(+/-) EML cells is associated with reduced levels of secondary granule protein (SGP) gene expression. Furthermore, we demonstrate a positive role for Gfi-1 in SGP expression, in that Gfi-1 binds to and up-regulates the promoter of neutrophil collagenase (an SGP gene), in cooperation with wild-type but not with mutant C/EBPepsilon. We hypothesize that decreased Gfi-1 levels in our SGD patient, together with the mutant C/EBPepsilon, block SGP expression, thereby contributing to the underlying etiology of the disease in our patient.

Title: Epigenetic regulation of protein-coding and microRNA genes by the Gfi1-interacting tumor suppressor PRDM5.
Abstract: Gfi1 transcriptionally governs hematopoiesis, and its mutations produce neutropenia. In an effort to identify Gfi1-interacting proteins and also to generate new candidate genes causing neutropenia, we performed a yeast two-hybrid screen with Gfi1. Among other Gfi1-interacting proteins, we identified a previously uncharacterized member of the PR domain-containing family of tumor suppressors, PRDM5. PRDM5 has 16 zinc fingers, and we show that it acts as a sequence-specific, DNA binding transcription factor that targets hematopoiesis-associated protein-coding and microRNA genes, including many that are also targets of Gfi1. PRDM5 epigenetically regulates transcription similarly to Gfi1: it recruits the histone methyltransferase G9a and class I histone deacetylases to its target gene promoters and demonstrates repressor activity on synthetic reporters; on endogenous target genes, however, it functions as an activator, in addition to a repressor. Interestingly, genes that PRDM5 activates, as opposed to those it represses, are also targets of Gfi1, suggesting a competitive mechanism through which two repressors could cooperate in order to become transcriptional activators. In neutropenic patients, we identified PRDM5 protein sequence variants perturbing transcriptional function, suggesting a potentially important role in hematopoiesis.

Title: Gfi1 ubiquitination and proteasomal degradation is inhibited by the ubiquitin ligase Triad1.
Abstract: Growth factor independence 1 (Gfi1) is a transcriptional repressor essential for the function and development of many different hematopoietic lineages. The Gfi1 protein expression is regulated by the ubiquitin-proteasome system. In granulocytes, Gfi1 is rapidly degraded by the proteasome, while it is more stable in monocytes. How the ubiquitination and degradation of Gfi1 is regulated is unclear. Here, we show that the ubiquitin ligase Triad1 interacts with the DNA-binding domain of Gfi1. Unexpectedly, we found that Triad1 inhibited Gfi1 ubiquitination, resulting in a prolonged half-life. Down-regulation of endogenous Triad1 by siRNAs resulted in increased Gfi1 ubiquitination. In U937 cells, Triad1 caused an increase in endogenous Gfi1 protein levels and slowed cell proliferation in a similar manner when Gfi1 itself was expressed. A Triad1 mutant that lacks the Gfi1-binding domain did not affect Gfi1 levels and proliferation. Because neither proteasome-ubiquitin nor Triad1 ubiquitin ligase activity was required for the inhibition of Gfi1 ubiquitination, these data suggest that Triad1 competes for Gfi1 binding with as yet to be identified E3 ubiquitin ligases that do mark Gfi1 for proteasomal degradation. The fine-tuning of Gfi1 protein levels regulated by Triad1 defines an unexpected role for this protein in hematopoiesis.

Title: Epigenetic regulation of hematopoietic differentiation by Gfi-1 and Gfi-1b is mediated by the cofactors CoREST and LSD1.
Abstract: Gfi-1 and Gfi-1b are homologous transcriptional repressors involved in diverse developmental contexts, including hematopoiesis and oncogenesis. Transcriptional repression by Gfi proteins requires the conserved SNAG domain. To elucidate the function of Gfi proteins, we purified Gfi-1b complexes and identified interacting proteins. Prominent among these is the corepressor CoREST, the histone demethylase LSD1, and HDACs 1 and 2. CoREST and LSD1 associate with Gfi-1/1b via the SNAG repression domain. Gfi-1b further recruits these cofactors to the majority of target gene promoters in vivo. Inhibition of CoREST and LSD1 perturbs differentiation of erythroid, megakaryocytic, and granulocytic cells as well as primary erythroid progenitors. LSD1 depletion derepresses Gfi targets in lineage-specific patterns, accompanied by enhanced histone 3 lysine 4 methylation at the respective promoters. Overall, we show that chromatin regulatory proteins CoREST and LSD1 mediate transcriptional repression by Gfi proteins. Lineage-restricted deployment of these cofactors through interaction with Gfi proteins controls hematopoietic differentiation.

Title: The Ajuba LIM domain protein is a corepressor for SNAG domain mediated repression and participates in nucleocytoplasmic Shuttling.
Abstract: The SNAG repression domain is comprised of a highly conserved 21-amino acid sequence, is named for its presence in the Snail/growth factor independence-1 class of zinc finger transcription factors, and is present in a variety of proto-oncogenic transcription factors and developmental regulators. The prototype SNAG domain containing oncogene, growth factor independence-1, is responsible for the development of T cell thymomas. The SNAIL proteins also encode the SNAG domain and play key roles in epithelial mesenchymal differentiation events during development and metastasis. Significantly, these oncogenic functions require a functional SNAG domain. The molecular mechanisms of SNAG domain-mediated transcriptional repression are largely unknown. Using a yeast two-hybrid strategy, we identified Ajuba, a multiple LIM domain protein that can function as a corepressor for the SNAG domain. Ajuba interacts with the SNAG domain in vitro and in vivo, colocalizes with it, and enhances SNAG-mediated transcriptional repression. Ajuba shuttles between the cytoplasm and the nucleus and may form a novel intracellular signaling system. Using an integrated reporter gene combined with chromatin immunoprecipitation, we observed rapid, SNAG-dependent assembly of a multiprotein complex that included Ajuba, SNAG, and histone modifications consistent with the repressed state. Thus, SNAG domain proteins may bind Ajuba, trapping it in the nucleus where it functions as an adapter or molecular scaffold for the assembly of macromolecular repression complexes at target promoters.

Title: CD34, RAB20, PU.1 and GFI1 mRNA expression in myelodysplastic syndrome.
Abstract: Myelodysplastic syndrome (MDS) with hypocellular bone marrow (BM) is often difficult to distinguish from aplastic anemia (AA). Furthermore, the diagnosis of MDS with low blast counts and normal karyotype may be problematic. These issues highlight the need for a reliable marker for the diagnosis of MDS. This study was conducted to determine if changes of mRNA expression in any of the four selected genes would be useful markers for differentiation of hypoplastic MDS from AA, and MDS from benign disease, as well as to investigate whether mRNA expressions differ between MDS risk subgroups. Thirty-five patients diagnosed with MDS, 27 patients with AA and 17 patients with benign diseases were included. The CD34, RAB20, PU.1 and GFI1 mRNA levels were measured by real-time RT-PCR. The CD34 mRNA expressions in hypoplastic MDS were higher than those found in AA. PU.1 and GFI1 mRNA expressions were significantly lower in MDS with low blast counts and normal karyotype than those of benign disease. High-risk MDS showed higher CD34 expressions than those of low-risk MDS. This study suggests that measurement of CD34 and GFI1 mRNA expressions could be useful as a diagnostic and prognostic marker for MDS.

Title: Gfi1-mediated stabilization of GATA3 protein is required for Th2 cell differentiation.
Abstract: The differentiation of naive CD4 T cells into Th2 cells requires the T cell receptor-mediated activation of the ERK MAPK cascade. Little is known, however, in regard to how the ERK MAPK cascade regulates Th2 cell differentiation. We herein identified Gfi1 (growth factor independent-1) as a downstream target of the ERK MAPK cascade for Th2 cell differentiation. In the absence of Gfi1, interleukin-5 production and the change of histone modification at the interleukin-5 gene locus were severely impaired. Furthermore, the interferon gamma gene showed a striking activation in the Gfi1(-/-) Th2 cells. An enhanced ubiquitin/proteasome-dependent degradation of GATA3 protein was observed in Gfi1(-/-) Th2 cells, and the overexpression of GATA3 eliminated the defect of Th2 cell function in Gfi1-deficient Th2 cells. These data suggest that the T cell receptor-mediated induction of Gfi1 controls Th2 cell differentiation through the regulation of GATA3 protein stability.

Title: Ajuba functions as a histone deacetylase-dependent co-repressor for autoregulation of the growth factor-independent-1 transcription factor.
Abstract: Growth factor independent-1 (Gfi1) is a zinc finger protein with a SNAG-transcriptional repressor domain. Ajuba is a LIM domain protein that shuttles between the cytoplasm and the nucleus. Ajuba functions as a co-repressor for synthetic Gfi1 SNAG-repressor domain-containing constructs, but a role for Ajuba co-repression of the cognate DNA bound Gfi1 protein has not been defined. Co-immunoprecipitation of synthetic and endogenous proteins and co-elution with gel filtration suggest that an endogenous Ajuba.Gfi1.HDAC multiprotein complex is possible. Active histone deacetylase activity co-immunoprecipitates with Ajuba or Gfi1, and both proteins depend upon histone deacetylases for full transcriptional repression activity. Ajuba LIM domains directly bind to Gfi1, but the association is not SNAG domain-dependent. ChIP analysis and reciprocal knockdown experiments suggest that Ajuba selectively functions as a co-repressor for Gfi1 autoregulation. The data suggest that Ajuba is utilized as a corepressor selectively on Gfi1 target genes.

Title: Loss of function genetic screens reveal MTGR1 as an intracellular repressor of beta1 integrin-dependent neurite outgrowth.
Abstract: Integrins are transmembrane receptors that promote neurite growth and guidance. To identify regulators of integrin-dependent neurite outgrowth, here we used two loss of function genetic screens in SH-SY5Y neuroblastoma cells. First, we screened a genome-wide retroviral library of genetic suppressor elements (GSEs). Among the many genes identified in the GSE screen, we isolated the hematopoetic transcriptional factor MTGR1 (myeloid translocation gene-related protein-1). Treatment of SH-SY5Y cells with MTGR1 siRNA enhanced neurite outgrowth and concurrently increased expression of GAP-43, a protein linked to neurite outgrowth. Second, we transduced SH-SY5Y with a genome-wide GFP-labeled lentiviral siRNA library, which expressed 40,000 independent siRNAs targeting 8500 human genes. From this screen we isolated GFI1 (growth factor independence-1), which, like MTGR1, is a member of the myeloid translocation gene on 8q22 (MTG8)/ETO protein complex of nuclear repressor proteins. These results reveal novel contributions of MTGR1 and GFI1 to the regulation of neurite outgrowth and identify novel repressors of integrin-dependent neurite outgrowth.

Title: Gfi-1 represses CDKN2B encoding p15INK4B through interaction with Miz-1.
Abstract: Gfi-1 is a nuclear zinc finger (ZF) transcriptional repressor that plays an important role in hematopoiesis and inner ear development, and has been implicated in lymphomagenesis. Gfi-1 represses transcription by directly binding to the consensus DNA sequence in the promoters of its target genes. We report here an alternative mechanism by which Gfi-1 represses CDKN2B encoding p15(INK4B). Gfi-1 does not directly bind to CDKN2B, but interacts with Miz-1 and, via Miz-1, is recruited to the core promoter of CDKN2B. Miz-1 is a POZ-ZF transcription factor that has been shown to mediate transcriptional repression by c-Myc. Like c-Myc, upon recruitment to the CDKN2B promoter, Gfi-1 represses transcriptional activation of CDKN2B by Miz-1 and in response to TGFbeta. Consistent with its role in repressing CDKN2B transcription, knockdown of Gfi-1 in human leukemic cells or deficiency of Gfi-1 in mouse bone marrow cells results in augmented expression of p15(INK4B). Notably, Gfi-1 and c-Myc are both recruited to the CDKN2B core promoter and act in collaboration to repress CDKN2B. Our data reveal a mechanism of transcriptional repression by Gfi-1 and may have important implications for understanding the roles of Gfi-1 in normal development and tumorigenesis.

Title: Contributions to neutropenia from PFAAP5 (N4BP2L2), a novel protein mediating transcriptional repressor cooperation between Gfi1 and neutrophil elastase.
Abstract: "Neutropenia" refers to deficient numbers of neutrophils, the most abundant type of white blood cell. Two main forms of inherited neutropenia are cyclic neutropenia, in which neutrophil counts oscillate with a 21-day frequency, and severe congenital neutropenia, in which static neutropenia may evolve at times into leukemia. Mutations of ELA2, encoding the protease neutrophil elastase, can cause both disorders. Among other genes, severe congenital neutropenia can also result from mutations affecting the transcriptional repressor Gfi1, one of whose genetic targets is ELA2, suggesting that the two act through similar mechanisms. In order to identify components of a common pathway regulating neutrophil production, we conducted yeast two-hybrid screens with Gfi1 and neutrophil elastase and detected a novel protein, PFAAP5 (also known as N4BP2L2), interacting with both. Expression of PFAAP5 allows neutrophil elastase to potentiate the repression of Gfi1 target genes, as determined by reporter assays, RNA interference, chromatin immunoprecipitation, and impairment of neutrophil differentiation in HSCs with PFAAP5 depletion, thus delineating a mechanism through which neutrophil elastase could regulate its own synthesis. Our findings are consistent with theoretical models of cyclic neutropenia proposing that its periodicity can be explained through disturbance of a feedback circuit in which mature neutrophils inhibit cell proliferation, thereby homeostatically regulating progenitor populations.

Title: The growth factor independence-1 (Gfi1) is overexpressed in chronic myelogenous leukemia.
Abstract: The activation of ABL tyrosine kinase in BCR/ABL-positive chronic myelogenous leukemia (CML) leads to a diversity of biological changes related to the pathogenesis of the disease. In CML patients, we determined the expression of growth factor independence-1 (Gfi1), a transcription repressor with weak oncogenic activity. Our data demonstrated that the Gfi1 mRNA level in both the mononuclear cells and purified CD34(+) cells from CML were significantly higher as measured by quantitative real-time PCR. Using flow cytometry and Western blot, we also showed that the Gfi1 protein content was increased in CML CD34(+) cells. The expression of Gfi1 was correlated with BCR/ABL significantly. Gfi1 may be implicated in the pathogenesis of CML and can serve as a potential target for the management of the disease.

Title: A variant allele of Growth Factor Independence 1 (GFI1) is associated with acute myeloid leukemia.
Abstract: The GFI1 gene encodes a transcriptional repressor, which regulates myeloid differentiation. In the mouse, Gfi1 deficiency causes neutropenia and an accumulation of granulomonocytic precursor cells that is reminiscent of a myelodysplastic syndrome. We report here that a variant allele of GFI1 (GFI1(36N)) is associated with acute myeloid leukemia (AML) in white subjects with an odds ratio of 1.6 (P &lt; 8 x 10(-5)). The GFI1(36N) variant occurred in 1806 AML patients with an allele frequency of 0.055 compared with 0.035 in 1691 healthy control patients in 2 independent cohorts. We observed that both GFI1 variants maintain the same activity as transcriptional repressors but differ in their regulation by the AML1/ETO (RUNX1/RUNX1T1) fusion protein produced in AML patients with a t(8;21) translocation. AML1/ETO interacts and colocalizes with the more common GFI1(36S) form in the nucleus and inhibits its repressor activity. However, the variant GFI1(36N) protein has a different subnuclear localization than GFI1(36S). As a consequence, AML1/ETO does not colocalize with GFI1(36N) and is unable to inhibit its repressor activity. We conclude that both variants of GFI1 differ in their ability to be regulated by interacting proteins and that the GFI1(36N) variant form exhibits distinct biochemical features that may confer a predisposition to AML.

Title: Tag-SNP analysis of the GFI1-EVI5-RPL5-FAM69 risk locus for multiple sclerosis.
Abstract: A recent genome-wide association study conducted by the International Multiple Sclerosis Genetic Consortium (IMSGC) identified, among others, a number of putative multiple sclerosis (MS) susceptibility variants at position 1p22. Twenty-one SNPs positively associated with MS were located at the GFI-EVI5-RPL5-FAM69A locus. In this study, we performed an analysis and fine mapping of this locus, genotyping eight Tag-SNPs in 732 MS patients and 974 controls from Spain. We observed an association with MS in three of eight Tag-SNPs: rs11804321 (P=0.008, OR=1.29; 95% CI=1.08-1.54), rs11808092 (P=0.048, OR=1.19; 95% CI=1.03-1.39) and rs6680578 (P=0.0082, OR=1.23; 95% CI=1.07-1.41). After correcting for multiple comparisons and using logistic regression analysis to test the addition of each SNP to the most associated SNPs, we observed that rs11804321 alone was sufficient to model the association. This Tag-SNP captures two SNPs in complete linkage disequilibrium (r(2)=1), both located within the 17th intron of the EVI5 gene. Our findings agree with the corresponding data of the recent IMSGC study and present new genetic evidence that points to EVI5 as a factor of susceptibility to MS.

Title: A role of Miz-1 in Gfi-1-mediated transcriptional repression of CDKN1A.
Abstract: Zinc-finger (ZF) transcriptional repressor growth factor independence 1 (Gfi-1) has an important role in hematopoiesis and inner ear development, and also functions as an oncoprotein that cooperates with c-Myc in lymphomagenesis. Gfi-1 represses transcription by directly binding to conserved sequences in the promoters of its target genes. CDKN1A encoding p21(Cip1) has been identified as a Gfi-1 target gene and has been shown to contain Gfi-1 binding sites in the upstream promoter region. We show here that Gfi-1 represses CDKN1A in a manner that is independent of its DNA binding activity. Gfi-1 interacts with POZ-ZF transcription factor Miz-1, originally shown to be a c-Myc-interacting partner, and through Miz-1 binds to the CDKN1A core promoter. Interestingly, Gfi-1 and c-Myc, through Miz-1, form a ternary complex on the CDKN1A promoter, and function in collaboration to repress CDKN1A. Gfi-1 knockdown results in enhanced levels of p21(Cip1) and attenuated cell proliferation. Notably, similar to c-Myc, the expression of Gfi-1 is downregulated by transforming growth factor-beta (TGFbeta) and the level of Gfi-1 influences the response of cells to the cytostatic effect of TGFbeta. Our data reveal an important mechanism by which Gfi-1 regulates cell proliferation and may also have implications for understanding the role of Gfi-1 in lymphomagenesis.

Title: Zinc finger protein Gfi1 controls the endotoxin-mediated Toll-like receptor inflammatory response by antagonizing NF-kappaB p65.
Abstract: Endotoxin (bacterial lipopolysaccharide [LPS]) causes fatal septic shock via the Toll-like receptor 4 (TLR-4) protein present on innate immunity effector cells, which activates nuclear factor kappa B (NF-kappaB), inducing proinflammatory cytokines, including tumor necrosis factor alpha (TNF-alpha). An early step in this process involves nuclear sequestration of the p65-RelA NF-kappaB subunit, enabling transcriptional activation of target inflammatory cytokine genes. Here, we analyzed the role of the nuclear zinc finger protein Gfi1 in the TLR response using primary bone marrow-derived macrophages. We show that upon LPS stimulation, expression of Gfi1 is induced with kinetics similar to those of nuclear translocation of p65 and that Gfi1 interacts with p65 and inhibits p65-mediated transcriptional transactivation by interfering with p65 binding to target gene promoter DNA. Gfi1-deficient macrophages show abnormally high mRNA levels of the TNF-alpha gene and many other p65 target genes and a higher rate of TNF promoter occupancy by p65 than wild-type cells after LPS stimulation, suggesting that Gfi1 functions as an antagonist of NF-kappaB activity at the level of promoter binding. Our findings identify a new function of Gfi1 as a general negative regulator of the endotoxin-initiated innate immune responses, including septic shock and possibly other severe inflammatory diseases.

Title: Gfi1-cells and circuits: unraveling transcriptional networks of development and disease.
Abstract: The review will integrate current knowledge of transcriptional circuits whose dysregulation leads to autoimmunity, neutropenia and leukemia.

Title: [Expression of growth-factor independence 1 in patients with leukemia and its significance].
Abstract: This study was purposed to investigate the expression of the growth-factor independence 1 (GFI1) in patients with leukemia and its clinical significance. Bone marrow mononuclear cells were obtained from 65 newly diagnosed leukemia patients including 24 acute myeloid leukemia (AML), 18 chronic myelogenous leukemia (CML), 6 acute lymphoblastic leukemia (ALL), 17 blast crisis of chronic myelogenous leukemia and 13 patients with iron deficiency anemia (IDA) were used as controls. The relative expression of gene gfi1 was detected by reverse transcriptase-polymerase chain reaction (RT-PCR) and taqman quantitative real-time reverse transcription polymerase chain reaction (QRT-PCR). The results showed that gene expression of gene gfi1 in leukemia patients was obviously higher than that in controls and the difference was statistically significant (p &lt; 0.01), in which the expression of gene gfi1 in newly diagnosed CML patients was higher than that in newly diagnosed AML, newly diagnosed ALL, CML-BCP patients and the difference was significant (p &lt; 0.01). Expression of gene gfi1 in lymphocytic blast crisis of CML was higher than that in nonlymphocytic blast crisis of CML, and the difference was significant. It is concluded that gene gfi1 may play an important role in leukemia, especially in CML incidence and progression. The high level expression of gene gfi1 may be participate in the development of lymphoma.

Title: [Gfi-1 expression in leukemia patients and inhibitory effects of lentiviral vector mediated silence of Gfi-1 gene on proliferation in K562 cells].
Abstract: This study was aimed to quantitatively detect the expression levels of gfi-1 gene in leukemia patients, and to investigate the effect of gfi-1 gene silenced by short hairpin RNA (shRNA) on proliferation of leukemia cell line K562. Quantitative real-time PCR and Western blot were used to detect the mRNA and protein expression levels of GFI-1 in newly diagnosed patients with leukemia. One pair of oligonucleotide sequences targeted at human gfi-1 mRNA were designed and synthesized. The annealed oligonucleotide fragments were subcloned into pLVTHM vector. Virus particles were collected when the control and shRNA vectors had been co-transfected with the psPAX2 packaging plasmid and the envelope plasmid pMD2 G into HEK-293T cells using Lipofectamine 2000. The K562 cells were transfused with 1 x 10⁶ recombinant lentivirus-transfusing units plus 6 microg/ml of polybrene. Rea-time PCR and Western blot were used to detect the expressions of gfi-1 and bax mRNA after lentivirus transfusion. CCK-8 assays was used to evaluate the proliferation potential of cells. The results showed that the gfi-1 expression level in all leukemia patients was significantly higher than that in normal group (p &lt; 0.05); the gfi-1 mRNA expression in chronic myeloid leukemia (CML) and acute myeloid leukemia (AML) or acute lymphoid leukemia (ALL) patients was significantly higher than that in normal group (p &lt; 0.05); but the difference of gfi-1 mRNA expression between AL and CML or ALL and AML was not significant. Notably, the gfi-1 mRNA expression level had a positive correlation with high white blood cell count of &gt; 20.0 x 10⁹/L (p &lt; 0.05). As was expected, the mRNA and protein level of gfi-1 was reduced significantly in K562 cells after lentivirus transfusion, whereas the mRNA and protein level of bax was upregulated. And CCK-8 assay showed that gfi-1 gene silencing can inhibit K562 proliferation. It is concluded that gfi-1 expression is upregulated in leukemia patients and may contribute to leukemogenesis. The gfi-1 specific shRNA mediated by lentivirus can effectively down-regulate the expression of gfi-1 and inhibit the proliferation of K562 cells, which lay a basis for further research on gene therapy in leukemia.

Title: The p150 subunit of the histone chaperone Caf-1 interacts with the transcriptional repressor Gfi1.
Abstract: Modification of histones is critically involved in regulating chromatin structure and gene expression. The zinc finger protein Gfi1 silences transcription by recruiting a complex of histone modifying enzymes such as LSD-1/CoRest and HDAC-1 to target gene promoters. Here we present evidence that Gfi1 forms a complex with the p150 subunit of the histone chaperone chromatin assembly factor-1 (Caf-1). Gfi1 and p150 interact at endogenous expression levels and co-localize in distinct sub-nuclear structures. We show that p150 enhances Gfi1-mediated transcriptional repression and that it occupies Gfi1 target gene promoters in transfected cells and primary murine T cells only in the presence of Gfi1. Finally, size exclusion chromatography shows a fraction of p150 to coelute with Gfi1, LSD-1 and HDAC-1 and thus provides evidence that p150 is part of the Gfi1 repression complex. Since p150 binds directly to histones H3 and H4, our findings suggest that p150 may link the DNA-bound Gfi1 repressor complex to histones enabling modifications required for transcriptional silencing.

Title: Phylogenetic-based propagation of functional annotations within the Gene Ontology consortium.
Abstract: The goal of the Gene Ontology (GO) project is to provide a uniform way to describe the functions of gene products from organisms across all kingdoms of life and thereby enable analysis of genomic data. Protein annotations are either based on experiments or predicted from protein sequences. Since most sequences have not been experimentally characterized, most available annotations need to be based on predictions. To make as accurate inferences as possible, the GO Consortium's Reference Genome Project is using an explicit evolutionary framework to infer annotations of proteins from a broad set of genomes from experimental annotations in a semi-automated manner. Most components in the pipeline, such as selection of sequences, building multiple sequence alignments and phylogenetic trees, retrieving experimental annotations and depositing inferred annotations, are fully automated. However, the most crucial step in our pipeline relies on software-assisted curation by an expert biologist. This curation tool, Phylogenetic Annotation and INference Tool (PAINT) helps curators to infer annotations among members of a protein family. PAINT allows curators to make precise assertions as to when functions were gained and lost during evolution and record the evidence (e.g. experimentally supported GO annotations and phylogenetic information including orthology) for those assertions. In this article, we describe how we use PAINT to infer protein function in a phylogenetic context with emphasis on its strengths, limitations and guidelines. We also discuss specific examples showing how PAINT annotations compare with those generated by other highly used homology-based methods.

Title: Gfi1 expressed in bone marrow stromal cells is a novel osteoblast suppressor in patients with multiple myeloma bone disease.
Abstract: Protracted inhibition of osteoblast (OB) differentiation characterizes multiple myeloma (MM) bone disease and persists even when patients are in long-term remission. However, the underlying pathophysiology for this prolonged OB suppression is unknown. Therefore, we developed a mouse MM model in which the bone marrow stromal cells (BMSCs) remained unresponsive to OB differentiation signals after removal of MM cells. We found that BMSCs from both MM-bearing mice and MM patients had increased levels of the transcriptional repressor Gfi1 compared with controls and that Gfi1 was a novel transcriptional repressor of the critical OB transcription factor Runx2. Trichostatin-A blocked the effects of Gfi1, suggesting that it induces epigenetic changes in the Runx2 promoter. MM-BMSC cell-cell contact was not required for MM cells to increase Gfi1 and repress Runx2 levels in MC-4 before OBs or naive primary BMSCs, and Gfi1 induction was blocked by anti-TNF-α and anti-IL-7 antibodies. Importantly, BMSCs isolated from Gfi1(-/-) mice were significantly resistant to MM-induced OB suppression. Strikingly, siRNA knockdown of Gfi1 in BMSCs from MM patients significantly restored expression of Runx2 and OB differentiation markers. Thus, Gfi1 may have an important role in prolonged MM-induced OB suppression and provide a new therapeutic target for MM bone disease.

Title: The human GFI136N variant induces epigenetic changes at the Hoxa9 locus and accelerates K-RAS driven myeloproliferative disorder in mice.
Abstract: The coding single nucleotide polymorphism GFI136N in the human gene growth factor independence 1 (GFI1) is present in 3%-7% of whites and increases the risk for acute myeloid leukemia (AML) by 60%. We show here that GFI136N, in contrast to GFI136S, lacks the ability to bind to the Gfi1 target gene that encodes the leukemia-associated transcription factor Hoxa9 and fails to initiate histone modifications that regulate HoxA9 expression. Consistent with this, AML patients heterozygous for the GFI136N variant show increased HOXA9 expression compared with normal controls. Using ChipSeq, we demonstrate that GFI136N specific epigenetic changes are also present in other genes involved in the development of AML. Moreover, granulomonocytic progenitors, a bone marrow subset from which AML can arise in humans and mice, show a proliferative expansion in the presence of the GFI136N variant. In addition, granulomonocytic progenitors carrying the GFI136N variant allele have altered gene expression patterns and differ in their ability to grow after transplantation. Finally, GFI136N can accelerate a K-RAS driven fatal myeloproliferative disease in mice. Our data suggest that the presence of a GFI136N variant allele induces a preleukemic state in myeloid precursors by deregulating the expression of Hoxa9 and other AML-related genes.

Title: Potential roles for Gfi1 in the pathogenesis and proliferation of glioma.
Abstract: Glioblastoma multiforme (GBM) is a major form of adult brain tumour with relatively poor prognosis and high mortality. Temozolomide (TMZ)-based chemotherapy following neurosurgery and radiotherapy has been suggested as the first line of treatment and is proven to effectively prolong overall survival and enhance patient quality of life. However, not all patients benefit from this treatment because of drug resistance. Even patients with TMZ-sensitive GBM may become resistant, which is partly due to the restoration of activity of the DNA repair enzyme O(6)-methylguanine-DNA-methyltransferase (MGMT); thus, patients cannot evade eventual tumour recurrence. The cellular activity of MGMT is the most important determinant of TMZ-resistance. However, some patients with a low level of activated MGMT are also TMZ-resistant. The aberrant expression of HOXA9, one of the 39 class I homeobox genes, is a marker of poor prognosis, and its level gradually increases with histologic malignant progression, shorter time to overall survival (OS) and free progression survival (FPS) in glioma patients, which further supports an oncogenic role for HOXA9 in gliomas. The HOXA9-PI3K signalling pathway is an important mechanism in GBM that is independent of MGMT promoter methylation status. The DNA binding sites of growth factor independent-1 (Gfi1) can overlap with the HOXA9 promoter through the "AATC" versus "GATT" core sequence. The competition for this binding site inhibits the expression of HOXA9 and induces different transcriptional outcomes, which suggests a new direction for investigation of the mechanism underlying targeted therapy of malignant gliomas.

Title: Growth factor independence 1 (Gfi1) as a regulator of p53 activity and a new therapeutical target for ALL.
Abstract: The transcriptional repressor Gfi1 can be a so-called "oncorequisite" factor that is required for the development and maintenance of lymphoid neoplasia, such as Acute Lymphoblastic Leukemia (ALL), but does not have a direct role in the ontogeny of the disease. The study supporting this role of Gfi1 (Khandanpour C, Phelan J, et al., Cancer Cell, 2013, 23:200-214) shows that inhibition of Gfi1 cannot only cure mice from ALL but also blocks the expansion of human primary ALL cells. The study concludes that this feature of Gfi1 can be exploited to improve current ALL therapies.

Title: Gfi-1 is the transcriptional repressor of SOCS1 in acute myeloid leukemia cells.
Abstract: Silencing of SOCS1, a TSG, has been detected in various malignancies, including AML. However, the underlying mechanism of SOCS1 inactivation remains elusive. In this study, we explored the role of histone methylation in SOCS1 expression in AML cells. By ChIP assay, we demonstrated that G9a and SUV39H1, two enzymes catalyzing H3K9 methylation, were physically associated with the SOCS1 promoter, and treatment with chaetocin, a histone methyltransferase inhibitor, suppressed H3K9 methylation on the SOCS1 promoter and enhanced SOCS1 expression. Furthermore, knockdown of G9a and SUV39H1 by siRNA could also induce SOCS1 expression. On the other hand, SOCS1 knockdown by shRNA eliminated chaetocin-induced cell apoptosis. To investigate further whether any transcription factor was involved in H3K9 methylation-related SOCS1 repression, we scanned the sequences of the SOCS1 gene promoter and found two binding sites for Gfi-1, a transcription repressor. By DNA pull-down and ChIP assays, we showed that Gfi-1 directly bound the SOCS1 promoter, and ectopic Gfi-1 expression suppressed STAT5-induced SOCS1 promoter activation. In contrast, Gfi-1 knockdown by shRNA enhanced SOCS1 expression and inhibited STAT5 expression. Moreover, the knockdown of G9a completely rescued the repressive effect of Gfi-1 on STAT5A-induced SOCS1 promoter activation. Collectively, our study indicates that the expression of Gfi-1 contributes to SOCS1 silencing in AML cells through epigenetic modification, and suppression of histone methyltransferase can provide new insight in AML therapy.

Title: GFI1 is repressed by p53 and inhibits DNA damage-induced apoptosis.
Abstract: GFI1 is a transcriptional repressor that plays a critical role in hematopoiesis and has also been implicated in lymphomagenesis. It is still poorly understood how GFI1 expression is regulated in the hematopoietic system. We show here that GFI1 transcription was repressed by the tumor suppressor p53 in hematopoietic cells. Knockdown of p53 resulted in increased GFI1 expression and abolished DNA damage-induced GFI1 downregulation. In contrast, GFI1 expression was reduced and its downregulation in response to DNA damage was rescued upon restoration of p53 function in p53-deficient cells. In luciferase reporter assays, wild type p53, but not a DNA binding-defective p53 mutant, repressed the GFI1 promoter. Chromatin immunoprecipitation (ChIP) assays demonstrated that p53 bound to the proximal region of the GFI1 promoter. Detailed mapping of the GFI1 promoter indicated that GFI1 core promoter region spanning from -33 to +6 bp is sufficient for p53-mediated repression. This core promoter region contains a putative p53 repressive response element, mutation of which abolished p53 binding to and repression of GFI1 promoter. Significantly, apoptosis induced by DNA damage was inhibited upon Gfi1 overexpression, but augmented following GFI1 knockdown. Our data establish for the first time that GFI1 is repressed by p53 and add to our understanding of the roles of GFI1 in normal hematopoiesis and lymphomagenesis.

Title: The MYB proto-oncogene suppresses monocytic differentiation of acute myeloid leukemia cells via transcriptional activation of its target gene GFI1.
Abstract: The MYB gene is a master regulator of hematopoiesis and contributes to leukemogenesis in several species including humans. Although it is clear that MYB can promote proliferation, suppress apoptosis and block differentiation, the identities of the MYB target genes that mediate these effects have only been partially elucidated. Several studies, including our own, have collectively identified substantial numbers of MYB target genes, including candidates for each of these activities; however, functional validation, particularly in the case of differentiation suppression, has lagged well behind. Here we show that GFI1, which encodes an important regulator of hematopoietic stem cell (HSC) function and granulocytic differentiation, is a direct target of MYB in myeloid leukemia cells. Chromatin immunoprecipitation and reporter studies identified a functional MYB-binding site in the promoter region of GFI, whereas ectopic expression and small hairpin RNA-mediated knockdown of MYB resulted in concomitant increases and decreases, respectively, in GFI1 expression. We also demonstrate that GFI1, like MYB, can block the induced monocytic differentiation of a human acute myeloid leukemia cell line, and most importantly, that GFI1 is essential for MYB's ability to block monocytic differentiation. Thus, we have identified a target of MYB that is a likely mediator of its myeloid differentiation-blocking activity, and which may also be involved in MYB's activities in regulating normal HSC function and myeloid differentiation.

Title: Lymphoid progenitor cells from childhood acute lymphoblastic leukemia are functionally deficient and express high levels of the transcriptional repressor Gfi-1.
Abstract: Acute lymphoblastic leukemia (ALL) is the most frequent malignancy of childhood. Substantial progress on understanding the cell hierarchy within ALL bone marrow (BM) has been recorded in the last few years, suggesting that both primitive cell fractions and committed lymphoid blasts with immature stem cell-like properties contain leukemia-initiating cells. Nevertheless, the biology of the early progenitors that initiate the lymphoid program remains elusive. The aim of the present study was to investigate the ability of lymphoid progenitors from B-cell precursor ALL BM to proliferate and undergo multilineage differentiation. By phenotype analyses, in vitro proliferation assays, and controlled culture systems, the lymphoid differentiation potentials were evaluated in BM primitive populations from B-cell precursor ALL pediatric patients. When compared to their normal counterparts, functional stem and progenitor cell contents were substantially reduced in ALL BM. Moreover, neither B nor NK or dendritic lymphoid-cell populations developed recurrently from highly purified ALL-lymphoid progenitors, and their proliferation and cell cycle status revealed limited proliferative capacity. Interestingly, a number of quiescence-associated transcription factors were elevated, including the transcriptional repressor Gfi-1, which was highly expressed in primitive CD34⁺ cells. Together, our findings reveal major functional defects in the primitive hematopoietic component of ALL BM. A possible contribution of high levels of Gfi-1 expression in the regulation of the stem/progenitor cell biology is suggested.

Title: The downregulation of GFI1 by the EZH2-NDY1/KDM2B-JARID2 axis and by human cytomegalovirus (HCMV) associated factors allows the activation of the HCMV major IE promoter and the transition to productive infection.
Abstract: Earlier studies had suggested that epigenetic mechanisms play an important role in the control of human cytomegalovirus (HCMV) infection. Here we show that productive HCMV infection is indeed under the control of histone H3K27 trimethylation. The histone H3K27 methyltransferase EZH2, and its regulators JARID2 and NDY1/KDM2B repress GFI1, a transcriptional repressor of the major immediate-early promoter (MIEP) of HCMV. Knocking down EZH2, NDY1/KDM2B or JARID2 relieves the repression and results in the upregulation of GFI1. During infection, the incoming HCMV rapidly downregulates the GFI1 mRNA and protein in both wild-type cells and in cells in which EZH2, NDY1/KDM2B or JARID2 were knocked down. However, since the pre-infection levels of GFI1 in the latter cells are significantly higher, the virus fails to downregulate it to levels permissive for MIEP activation and viral infection. Following the EZH2-NDY1/KDM2B-JARID2-independent downregulation of GFI1 in the early stages of infection, the virus also initiates an EZH2-NDY1/ΚDM2Β-JARID2-dependent program that represses GFI1 throughout the infection cycle. The EZH2 knockdown also delays histone H3K27 trimethylation in the immediate early region of HCMV, which is accompanied by a drop in H3K4 trimethylation that may contribute to the shEZH2-mediated repression of the major immediate early HCMV promoter. These data show that HCMV uses multiple mechanisms to allow the activation of the HCMV MIEP and to prevent cellular mechanisms from blocking the HCMV replication program.

Title: Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma.
Abstract: Medulloblastoma is a highly malignant paediatric brain tumour currently treated with a combination of surgery, radiation and chemotherapy, posing a considerable burden of toxicity to the developing child. Genomics has illuminated the extensive intertumoral heterogeneity of medulloblastoma, identifying four distinct molecular subgroups. Group 3 and group 4 subgroup medulloblastomas account for most paediatric cases; yet, oncogenic drivers for these subtypes remain largely unidentified. Here we describe a series of prevalent, highly disparate genomic structural variants, restricted to groups 3 and 4, resulting in specific and mutually exclusive activation of the growth factor independent 1 family proto-oncogenes, GFI1 and GFI1B. Somatic structural variants juxtapose GFI1 or GFI1B coding sequences proximal to active enhancer elements, including super-enhancers, instigating oncogenic activity. Our results, supported by evidence from mouse models, identify GFI1 and GFI1B as prominent medulloblastoma oncogenes and implicate 'enhancer hijacking' as an efficient mechanism driving oncogene activation in a childhood cancer.

Title: MiR-495 is a Predictive Biomarker that  Downregulates GFI1 Expression in  Medulloblastoma.
Abstract: Alterations in the expression level of miR-495 were recently observed in various tumours. Medulloblastoma is the most common malignant brain tumour in children. However, the clinical significance of miR-495 in medulloblastomas remains unclear.

Title: From cytopenia to leukemia: the role of Gfi1 and Gfi1b in blood formation.
Abstract: The DNA-binding zinc finger transcription factors Gfi1 and Gfi1b were discovered more than 20 years ago and are recognized today as major regulators of both early hematopoiesis and hematopoietic stem cells. Both proteins function as transcriptional repressors by recruiting histone-modifying enzymes to promoters and enhancers of target genes. The establishment of Gfi1 and Gfi1b reporter mice made it possible to visualize their cell type-specific expression and to understand their function in hematopoietic lineages. We now know that Gfi1 is primarily important in myeloid and lymphoid differentiation, whereas Gfi1b is crucial for the generation of red blood cells and platelets. Several rare hematologic diseases are associated with acquired or inheritable mutations in the GFI1 and GFI1B genes. Certain patients with severe congenital neutropenia carry mutations in the GFI1 gene that lead to the disruption of the C-terminal zinc finger domains. Other mutations have been found in the GFI1B gene in families with inherited bleeding disorders. In addition, the Gfi1 locus is frequently found to be a proviral integration site in retrovirus-induced lymphomagenesis, and new, emerging data suggest a role of Gfi1 in human leukemia and lymphoma, underlining the role of both factors not only in normal hematopoiesis, but also in a wide spectrum of human blood diseases.

Title: GFI1(36N) as a therapeutic and prognostic marker for myelodysplastic syndrome.
Abstract: Inherited gene variants play an important role in malignant diseases. The transcriptional repressor growth factor independence 1 (GFI1) regulates hematopoietic stem cell (HSC) self-renewal and differentiation. A single-nucleotide polymorphism of GFI1 (rs34631763) generates a protein with an asparagine (N) instead of a serine (S) at position 36 (GFI1(36N)) and has a prevalence of 3%-5% among Caucasians. Because GFI1 regulates myeloid development, we examined the role of GFI1(36N) on the course of MDS disease. To this end, we determined allele frequencies of GFI1(36N) in four independent MDS cohorts from the Netherlands and Belgium, Germany, the ICGC consortium, and the United States. The GFI1(36N) allele frequency in the 723 MDS patients genotyped ranged between 9% and 12%. GFI1(36N) was an independent adverse prognostic factor for overall survival, acute myeloid leukemia-free survival, and event-free survival in a univariate analysis. After adjustment for age, bone marrow blast percentage, IPSS score, mutational status, and cytogenetic findings, GFI1(36N) remained an independent adverse prognostic marker. GFI1(36S) homozygous patients exhibited a sustained response to treatment with hypomethylating agents, whereas GFI1(36N) patients had a poor sustained response to this therapy. Because allele status of GFI1(36N) is readily determined using basic molecular techniques, we propose inclusion of GFI1(36N) status in future prospective studies for MDS patients to better predict prognosis and guide therapeutic decisions.

Title: [Cytokine-mediated regulation of expression of Gfi1 and U2afll4 genes activated by T-cells with different differentiation status in vitro].
Abstract: 

Title: Genetic contribution to variation in DNA methylation at maternal smoking-sensitive loci in exposed neonates.
Abstract: Epigenome-wide DNA methylation association studies have identified highly replicable genomic loci sensitive to maternal smoking during gestation. The role of inter-individual genetic variation in influencing DNA methylation, leading to the possibility of confounding or bias of such associations, has not been assessed. We investigated whether the DNA methylation levels at the top 10 CpG sites previously associated with exposure to maternal smoking during gestation were associated with individual genetic variation at the genome-wide level. Genome-wide association tests between DNA methylation at the top 10 candidate CpG and genome-wide SNPs were performed in 736 case and control participants of the California Childhood Leukemia Study. Three of the strongest maternal-smoking sensitive CpG sites in newborns were significantly associated with SNPs located proximal to each gene: cg18146737 in the GFI1 gene with rs141819830 (P = 8.2×10<sup>-44</sup>), cg05575921 in the AHRR gene with rs148405299 (P = 5.3×10<sup>-10</sup>), and cg12803068 in the MYO1G gene with rs61087368 (P = 1.3×10<sup>-18</sup>). For the GFI1 CpG cg18146737, the underlying genetic variation at rs141819830 confounded the association between maternal smoking and DNA methylation in our data (the regression coefficient changed from -0.02 [P = 0.139] to -0.03 [P = 0.015] after including the genotype). Our results suggest that further studies using DNA methylation at cg18146737, cg05575921, or cg12803068 that aim to assess exposure to maternal smoking during gestation should include genotype at the corresponding SNP. New methods are required for adequate and routine inclusion of genotypic influence on DNA methylation in epigenome-wide association studies to control for potential confounding.

Title: GFI1 functions in transcriptional control and cell fate determination require SNAG domain methylation to recruit LSD1.
Abstract: Proper hematopoietic cell fate decisions require co-ordinated functions of transcription factors, their associated co-regulators, and histone-modifying enzymes. Growth factor independence 1 (GFI1) is a zinc finger transcriptional repressor and master regulator of normal and malignant hematopoiesis. While several GFI1-interacting proteins have been described, how GFI1 leverages these relationships to carry out transcriptional repression remains unclear. Here, we describe a functional axis involving GFI1, SMYD2, and LSD1 that is a critical contributor to GFI1-mediated transcriptional repression. SMYD2 methylates lysine-8 (K8) within a -(8)KSKK(11)- motif embedded in the GFI1 SNAG domain. Methylation-defective GFI1 SNAG domain lacks repressor function due to failure of LSD1 recruitment and persistence of promoter H3K4 di-methyl marks. Methylation-defective GFI1 also fails to complement GFI1 depletion phenotypes in developing zebrafish and lacks pro-growth and survival functions in lymphoid leukemia cells. Our data show a discrete methylation event in the GFI1 SNAG domain that facilitates recruitment of LSD1 to enable transcriptional repression and co-ordinate control of hematopoietic cell fate in both normal and malignant settings.

Title: TNIP2 is a Hub Protein in the NF-κB Network with Both Protein and RNA Mediated Interactions.
Abstract: The NF-κB family of transcription factors is pivotal in controlling cellular responses to environmental stresses; abnormal nuclear factor kappa-light-chain-enhancer of activated B cells (NF-κB) signaling features in many autoimmune diseases and cancers. Several components of the NF-κB signaling pathway have been reported to interact with the protein TNIP2 (also known as ABIN2), and TNIP2 can both positively and negatively regulate NF-κB- dependent transcription of target genes. However, the function of TNIP2 remains elusive and the cellular machinery associating with TNIP2 has not been systematically defined. Here we first used a broad MudPIT/Halo Affinity Purification Mass Spectrometry (AP-MS) approach to map the network of proteins associated with the NF-κB transcription factors, and establish TNIP2 as an NF-κB network hub protein. We then combined AP-MS with biochemical approaches in a more focused study of truncated and mutated forms of TNIP2 to map protein associations with distinct regions of TNIP2. NF-κB interacted with the N-terminal region of TNIP2. A central region of TNIP2 interacted with the endosomal sorting complex ESCRT-I via its TSG101 subunit, a protein essential for HIV-1 budding, and a single point mutant in TNIP2 disrupted this interaction. The major gene ontology category for TNIP2 associated proteins was mRNA metabolism, and several of these associations, like KHDRBS1, were lost upon depletion of RNA. Given the major association of TNIP2 with mRNA metabolism proteins, we analyzed the RNA content of affinity purified TNIP2 using RNA-Seq. Surprisingly, a specific limited number of mRNAs was associated with TNIP2. These RNAs were enriched for transcription factor binding, transcription factor cofactor activity, and transcription regulator activity. They included mRNAs of genes in the Sin3A complex, the Mediator complex, JUN, HOXC6, and GATA2. Taken together, our findings suggest an expanded role for TNIP2, establishing a link between TNIP2, cellular transport machinery, and RNA transcript processing.

Title: Sudden infant death syndrome: exposure to cigarette smoke leads to hypomethylation upstream of the growth factor independent 1 (GFI1) gene promoter.
Abstract: Smoking during pregnancy has long been known as an important risk factor for sudden infant death syndrome (SIDS). However, the precise relationship between the smoking behavior of the mother and SIDS still remains unclear. In this study, the influence of prenatal smoking exposure on the childrens' DNA methylation state of a CpG island located upstream of the promoter of the growth factor independent 1 (GFI1) gene was analyzed.

Title: Simvastatin attenuates macrophage-mediated gemcitabine resistance of pancreatic ductal adenocarcinoma by regulating the TGF-β1/Gfi-1 axis.
Abstract: Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy with an intrinsic resistance to almost all chemotherapeutic drugs, including gemcitabine. An abundance of tumor-associated macrophages (TAMs), which can promote the resistance of PDAC to gemcitabine, has been observed in the microenvironments of several tumors. In this study, we confirmed that incubation in TAM-conditioned medium (TAM-CM) reduces the gemcitabine-induced apoptosis of PDAC cells. Simvastatin attenuated the TAM-mediated resistance of PDAC cells to gemcitabine. Further investigation found that simvastatin reversed the TAM-mediated down-regulation of Gfi-1 and up-regulation of CTGF and HMGB1. Simvastatin induced Gfi-1 expression, which increased the sensitivity of PDAC cells to gemcitabine by decreasing TGF-β1 secretion by TAMs. A luciferase reporter assay and ChIP assay revealed that Gfi-1 directly repressed the transcription of CTGF and HMGB1. Simvastatin also reversed the effects of gemcitabine on the expression of TGF-β1 and Gfi-1 and reduced the resistance of PDAC to gemcitabine in vivo. These results provide the first evidence that simvastatin attenuates the TAM-mediated gemcitabine resistance of PDAC by blocking the TGF-β1/Gfi-1 axis. These findings suggest the TGF-β1/Gfi-1 axis as a novel therapeutic target for treating PDAC.

Title: GFI1 downregulation promotes inflammation-linked metastasis of colorectal cancer.
Abstract: Inflammation is frequently associated with initiation, progression, and metastasis of colorectal cancer (CRC). Here, we unveil a CRC-specific metastatic programme that is triggered via the transcriptional repressor, GFI1. Using data from a large cohort of clinical samples including inflammatory bowel disease and CRC, and a cellular model of CRC progression mediated by cross-talk between the cancer cell and the inflammatory microenvironment, we identified GFI1 as a gating regulator responsible for a constitutively activated signalling circuit that renders CRC cells competent for metastatic spread. Further analysis of mouse models with metastatic CRC and human clinical specimens reinforced the influence of GFI1 downregulation in promoting CRC metastatic spread. The novel role of GFI1 is uncovered for the first time in a human solid tumour such as CRC. Our results imply that GFI1 is a potential therapeutic target for interfering with inflammation-induced CRC progression and spread.

Title: miRNA-650 exerts anti-leukemia activity by inhibiting cell proliferation through Gfi1 targeting.
Abstract: Acute myeloid leukemia (AML) is the most common malignancy of the bone marrow with a high mortality. Recent advances in high-throughput sequencing have led to the identification of various miRNAs implicated in the pathogenesis of AML. We found in this study that miR-650, a miRNA that was traditionally considered to participate in the onset of hepatocellular carcinoma, might play a significant role in AML development and progression.

Title: Locus-specific DNA methylation in the placenta is associated with levels of pro-inflammatory proteins in cord blood and they are both independently affected by maternal smoking during pregnancy.
Abstract: We investigated the impact of maternal smoking during pregnancy on placental DNA methylation and how this may mediate the association between maternal smoking and pro-inflammatory proteins in cord blood. The study population consisted of 27 individuals exposed to maternal smoking throughout pregnancy, 32 individuals exposed during a proportion of the pregnancy, and 61 unexposed individuals. Methylation of 11 regions within 6 genes in placenta tissue was assessed by pyrosequencing. Levels of 7 pro-inflammatory proteins in cord blood were assessed by electrochemiluminescence. Differential methylation was observed in the CYP1A1 promoter and AHRR gene body regions between women who smoked throughout pregnancy and non-smokers on the fetal-side of the placenta and in the GFI1 promoter between women who quit smoking while pregnant and non-smokers on the maternal-side of the placenta. Maternal smoking resulted in elevated levels of IL-8 protein in cord blood, which was not mediated by DNA methylation of our candidate regions at either the maternal or the fetal side of the placenta. Placental DNA methylation was associated with levels of inflammatory proteins in cord blood. Our observations suggest that maternal smoking during pregnancy affects both placental DNA methylation and the neonate's immune response.

Title: Prognostic significance of high GFI1 expression in AML of normal karyotype and its association with a FLT3-ITD signature.
Abstract: Growth Factor Independence 1 (GFI1) is a transcriptional repressor that plays a critical role during both myeloid and lymphoid haematopoietic lineage commitment. Several studies have demonstrated the involvement of GFI1 in haematological malignancies and have suggested that low expression of GFI1 is a negative indicator of disease progression for both myelodysplastic syndromes (MDS) and acute myeloid leukaemia (AML). In this study, we have stratified AML patients into those defined as having a normal karyotype (CN-AML). Unlike the overall pattern in AML, those patients with CN-AML have a poorer survival rate when GFI1 expression is high. In this group, high GFI1 expression is paralleled by higher FLT3 expression, and, even when the FLT3 gene is not mutated, exhibit a FLT3-ITD signature of gene expression. Knock-down of GFI1 expression in the human AML Fujioka cell line led to a decrease in the level of FLT3 RNA and protein and to the down regulation of FLT3-ITD signature genes, thus linking two major prognostic indicators for AML.

Title: Enforced GFI1 expression impedes human and murine leukemic cell growth.
Abstract: The differentiation of haematopoietic cells is regulated by a plethora of so-called transcription factors (TFs). Mutations in genes encoding TFs or graded reduction in their expression levels can induce the development of various malignant diseases such as acute myeloid leukaemia (AML). Growth Factor Independence 1 (GFI1) is a transcriptional repressor with key roles in haematopoiesis, including regulating self-renewal of haematopoietic stem cells (HSCs) as well as myeloid and lymphoid differentiation. Analysis of AML patients and different AML mouse models with reduced GFI1 gene expression levels revealed a direct link between low GFI1 protein level and accelerated AML development and inferior prognosis. Here, we report that upregulated expression of GFI1 in several widely used leukemic cell lines inhibits their growth and decreases the ability to generate colonies in vitro. Similarly, elevated expression of GFI1 impedes the in vitro expansion of murine pre-leukemic cells. Using a humanized AML model, we demonstrate that upregulation of GFI1 expression leads to myeloid differentiation morphologically and immunophenotypically, increased level of apoptosis and reduction in number of cKit<sup>+</sup> cells. These results suggest that increasing GFI1 level in leukemic cells with low GFI1 expression level could be a therapeutic approach.

Title: GFI1 facilitates efficient DNA repair by regulating PRMT1 dependent methylation of MRE11 and 53BP1.
Abstract: GFI1 is a transcriptional regulator expressed in lymphoid cells, and an "oncorequisite" factor required for development and maintenance of T-lymphoid leukemia. GFI1 deletion causes hypersensitivity to ionizing radiation, for which the molecular mechanism remains unknown. Here, we demonstrate that GFI1 is required in T cells for the regulation of key DNA damage signaling and repair proteins. Specifically, GFI1 interacts with the arginine methyltransferase PRMT1 and its substrates MRE11 and 53BP1. We demonstrate that GFI1 enables PRMT1 to bind and methylate MRE11 and 53BP1, which is necessary for their function in the DNA damage response. Thus, our results provide evidence that GFI1 can adopt non-transcriptional roles, mediating the post-translational modification of proteins involved in DNA repair. These findings have direct implications for treatment responses in tumors overexpressing GFI1 and suggest that GFI1's activity may be a therapeutic target in these malignancies.

Title: MiR-650 regulates the proliferation, migration and invasion of human oral cancer by targeting growth factor independent 1 (Gfi1).
Abstract: Oral cancer being one of the lethal cancers is generally detected at advanced stages and causes significant mortality world over. The unavailability of the reliable biomarkers and therapeutic targets/agents forms a bottleneck in the treatment of oral cancer. MicroRNAs are considered of immense therapeutic potential for the treatment of cancer. Consistently, in this study the role and therapeutic potential of miR-650 was explored in oral cancer. The analysis of miR-650 expression by qRT-PCR revealed significant (p &lt; 0.05) upregulation of miR-650 in oral cancer cell lines. Cell cycle analysis by flow cytometery revealed that suppression of miR-650 significantly (p &lt; 0.05) inhibits the proliferation of the SCC-25 cells by prompting Sub-G1 cell cycle arrest. Further, miR-650 suppression also inhibited the migration and invasion of the SCC-25 oral cancer cells as revealed by transwell assays. TargetScan analysis showed that miR-650 targets Growth factor independent 1 (Gfi1). Moreover, the results of western blot analysis showed that miR-650 suppression inhibits the expression of Gfi1. Interestingly, suppression of Gfi1 exhibited similar effects on cell proliferation, migration and invasion of the oral cancer cells as that of miR-650 suppression. Nonetheless, miR-650 promoted the proliferation, migration and invasion of the SCC-25 cells by upregulating the expression of Gfi1. Moreover, overexpression of miR-650 could not rescue the effects of Gfi1 silencing on SCC-25 oral cancer cells. Conversely, overexpression of Gfi1 could rescue the effects of miR-650 inhibition on SCC-25 cell proliferation, migration and invasion. Additionally, miR-650 suppression could also inhibit the xenografted tumor growth in vivo by inhibiting the expression of Gfi1. Taken together, miR-650 may prove to be an important therapeutic target for the management of oral cancers.

Title: Association of maternal prenatal smoking GFI1-locus and cardio-metabolic phenotypes in 18,212 adults.
Abstract: DNA methylation at the GFI1-locus has been repeatedly associated with exposure to smoking from the foetal period onwards. We explored whether DNA methylation may be a mechanism that links exposure to maternal prenatal smoking with offspring's adult cardio-metabolic health.
```

## Considerations
scripts are parsing web pages (probably a bad idea), they could break if source page code changes
