# ncbi_scripts
simple scripts for command line querying ncbi resources like pubmed and clinvar via various inputs

## Usage
`head` each script for clarification on usage

```
$ head gene_abstracts.sh
### Usage
# bash gene_abstracts.sh $GENE
# where $GENE is a HUGO style gene name like GFI1, TP53, BRCA1, DDX17, etc
# Example: bash gene_abstracts.sh BRCA1
```

## Examples
get titles and abstracts for the gene EBF3 using `gene_abstracts.sh`

```
$ bash gene_abstracts.sh EBF3
>>> EBF3

Title: Initial assessment of human gene diversity and expression patterns based upon 83 million nucleotides of cDNA sequence.
Abstract: In an effort to identify new genes and analyse their expression patterns, 174,472 partial complementary DNA sequences (expressed sequence tags (ESTs)), totalling more than 52 million nucleotides of human DNA sequence, have been generated from 300 cDNA libraries constructed from 37 distinct organs and tissues. These ESTs have been combined with an additional 118,406 ESTs from the database dbEST, for a total of 83 million nucleotides, and treated as a shotgun sequence assembly project. The assembly process yielded 29,599 distinct tentative human consensus (THC) sequences and 58,384 non-overlapping ESTs. Of these 87,983 distinct sequences, 10,214 further characterize previously known genes based on statistically significant similarity to sequences in the available databases; the remainder identify previously unknown genes. Thirty tissues were sampled by over 1,000 ESTs each; only eight genes were matched by ESTs from all 30 tissues, and 227 genes were represented in 20 or more of the tissues sampled with more than 1,000 ESTs. Approximately 40% of identified human genes appear to be associated with basic energy metabolism, cell structure, homeostasis and cell division, 22% with RNA and protein synthesis and processing, and 12% with cell signalling and communication.

Title: Cloning and functional characterization of Roaz, a zinc finger protein that interacts with O/E-1 to regulate gene expression: implications for olfactory neuronal development.
Abstract: We have identified a protein, Rat O/E-1-associated zinc finger protein (Roaz), that plays a role in regulating the temporal and spatial pattern of olfactory neuronal-specific gene expression. This protein functions by interacting with the olfactory factor O/E-1 and modulating its transcriptional activity. Roaz, isolated via a yeast two-hybrid screen, encoded a protein containing 29 C2H2 zinc fingers of the TFIIIA type. The Roaz mRNA was found in brain, eye, olfactory epithelium, spleen, and heart. In situ hybridization data indicated that Roaz was expressed in the basal layer, consisting of neural precursor cells and immature sensory neurons of the olfactory epithelium, but not in the mature receptor cells. We showed that the Roaz protein bound specifically to O/E-1 by using the yeast two-hybrid system. The two proteins formed a stable complex in coimmunoprecipitation and in vitro binding assays. Introduction of Roaz and O/E-1 into cells containing an olfactory promoter-driven luciferase reporter demonstrated that Roaz abolished O/E-1-mediated transcriptional activation. We propose that the function of Roaz is to modulate negatively the transactivational activity of O/E-1 and to act as a switch protein in the coordination of olfactory sensory neuron differentiation.

Title: Integrated genomic and epigenomic analyses pinpoint biallelic gene inactivation in tumors.
Abstract: Aberrant methylation of CpG islands and genomic deletion are two predominant mechanisms of gene inactivation in tumorigenesis, but the extent to which they interact is largely unknown. The lack of an integrated approach to study these mechanisms has limited the understanding of tumor genomes and cancer genes. Restriction landmark genomic scanning (RLGS; ref. 1) is useful for global analysis of aberrant methylation of CpG islands, but has not been amenable to alignment with deletion maps because the identity of most RLGS fragments is unknown. Here, we determined the nucleotide sequence and exact chromosomal position of RLGS fragments throughout the genome using the whole chromosome of origin of the fragments and in silico restriction digestion of the human genome sequence. To study the interaction of these gene-inactivation mechanisms in primary brain tumors, we integrated RLGS-based methylation analysis with high-resolution deletion maps from microarray-based comparative genomic hybridization (array CGH; ref. 3). Certain subsets of gene-associated CpG islands were preferentially affected by convergent methylation and deletion, including genes that exhibit tumor-suppressor activity, such as CISH1 (encoding SOCS1; ref. 4), as well as genes such as COE3 that have been missed by traditional non-integrated approaches. Our results show that most aberrant methylation events are focal and independent of deletions, and the rare convergence of these mechanisms can pinpoint biallelic gene inactivation without the use of positional cloning.

Title: Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.
Abstract: The National Institutes of Health Mammalian Gene Collection (MGC) Program is a multiinstitutional effort to identify and sequence a cDNA clone containing a complete ORF for each human and mouse gene. ESTs were generated from libraries enriched for full-length cDNAs and analyzed to identify candidate full-ORF clones, which then were sequenced to high accuracy. The MGC has currently sequenced and verified the full ORF for a nonredundant set of &gt;9,000 human and &gt;6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and 3,500 mouse genes also have been identified. All MGC sequences and clones are available without restriction through public databases and clone distribution networks (see http:mgc.nci.nih.gov).

Title: Complete sequencing and characterization of 21,243 full-length human cDNAs.
Abstract: As a base for human transcriptome and functional genomics, we created the "full-length long Japan" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.

Title: The DNA sequence and comparative analysis of human chromosome 10.
Abstract: The finished sequence of human chromosome 10 comprises a total of 131,666,441 base pairs. It represents 99.4% of the euchromatic DNA and includes one megabase of heterochromatic sequence within the pericentromeric region of the short and long arm of the chromosome. Sequence annotation revealed 1,357 genes, of which 816 are protein coding, and 430 are pseudogenes. We observed widespread occurrence of overlapping coding genes (either strand) and identified 67 antisense transcripts. Our analysis suggests that both inter- and intrachromosomal segmental duplications have impacted on the gene count on chromosome 10. Multispecies comparative analysis indicated that we can readily annotate the protein-coding genes with current resources. We estimate that over 95% of all coding exons were identified in this study. Assessment of single base changes between the human chromosome 10 and chimpanzee sequence revealed nonsense mutations in only 21 coding genes with respect to the human sequence.

Title: An EBF3-mediated transcriptional program that induces cell cycle arrest and apoptosis.
Abstract: In a genome-wide screen for putative tumor suppressor genes, the EBF3 locus on the human chromosome 10q26.3 was found to be deleted or methylated in 73% of the examined cases of brain tumors. EBF3 is expressed in normal brain but is silenced in brain tumors. Therefore, it is suggested that EBF3 is a tumor suppressor. However, it remains unknown whether inactivation of EBF3 locus also occurs in other types of tumors and what functions of EBF3 underlie EBF3-mediated tumor suppression. We show here that expression of EBF3 resulted in cell cycle arrest and apoptosis. The expression of cyclin-dependent kinase inhibitors was profoundly affected with early activation and then repression of p21(cip1/waf1) and persistent activation of both p27(kip1) and p57(kip2), whereas genes involved in cell survival and proliferation were suppressed. EBF3 bound directly to p21(cip1/waf1) promoter and regulated transcription from both p21(cip1/waf1) and p27(kip1) promoters in reporter assays. Apoptosis occurred 48 hours after EBF3 expression with caspase-3 activation. Silencing of the EBF3 locus was observed in brain, colorectal, breast, liver, and bone tumor cell lines and its reactivation was achieved on treatment with 5-aza-2'-deoxycytidine and trichostatin A in a significant portion of these tumor cells. Therefore, EBF3 regulates a transcriptional program underlying a putative tumor suppression pathway.

Title: Large-scale mapping of human protein-protein interactions by mass spectrometry.
Abstract: Mapping protein-protein interactions is an invaluable tool for understanding protein function. Here, we report the first large-scale study of protein-protein interactions in human cells using a mass spectrometry-based approach. The study maps protein interactions for 338 bait proteins that were selected based on known or suspected disease and functional associations. Large-scale immunoprecipitation of Flag-tagged versions of these proteins followed by LC-ESI-MS/MS analysis resulted in the identification of 24,540 potential protein interactions. False positives and redundant hits were filtered out using empirical criteria and a calculated interaction confidence score, producing a data set of 6463 interactions between 2235 distinct proteins. This data set was further cross-validated using previously published and predicted human protein interactions. In-depth mining of the data set shows that it represents a valuable source of novel protein-protein interactions with relevance to human diseases. In addition, via our preliminary analysis, we report many novel protein interactions and pathway associations.

Title: Frequently methylated tumor suppressor genes in head and neck squamous cell carcinoma.
Abstract: Head and neck squamous cell carcinoma (HNSCC) is a very aggressive cancer. In advanced stages, the patient has poor chances of receiving effective treatment, and survival rates are low. To facilitate timely diagnosis and improve treatment, elucidation of early detection markers is crucial. DNA methylation markers are particularly advantageous because DNA methylation is an early event in tumorigenesis, and the epigenetic modification, 5-methylcytosine, is a stable mark. A genome-wide screen using Restriction Landmark Genomic Scanning found a set of genes that are most commonly methylated in head and neck cancers. Five candidate genes: septin 9 (SEPT9), sodium-coupled monocarboxylate transporter 1 (SLC5A8), functional smad-suppressing element on chromosome 18 (FUSSEL18), early B-cell factor 3 (EBF3), and iroquois homeobox 1 (IRX1) were methylated in 27% to 67% of the HNSCC patient samples tested. Furthermore, approximately 50% of the methylated tumor samples shared methylation between two of the five genes (most commonly between EBF3 and IRX1), and 15% shared methylation between three of the five genes. Expression analysis revealed candidate gene down-regulation in 25% to 93% of the HNSCC samples, and 5-aza-2'-deoxycytidine treatment was able to restore expression in at least 2 of 5 HNSCC cell lines for all of the genes tested. Overexpression of the three most frequently down-regulated candidates, SLC5A8, IRX1, and EBF3, validated their tumor suppressor potential by growth curve analysis and colony formation assay. Interestingly, all of the candidates identified may be involved in the transforming growth factor beta signaling pathway, which is often disrupted in HNSCC.

Title: [Construction of eukaryotic expression vector for EBF3 and EGFP fusion protein and its expression in HepG2 cells].
Abstract: To construct the eukaryotic expression vector for early human B-cell factor 3(EBF3) fused with the enhanced green fluorescent protein (EGFP) and express the fusion protein EBF3-EGFP in HepG2 cells.

Title: A census of human transcription factors: function, expression and evolution.
Abstract: Transcription factors are key cellular components that control gene expression: their activities determine how cells function and respond to the environment. Currently, there is great interest in research into human transcriptional regulation. However, surprisingly little is known about these regulators themselves. For example, how many transcription factors does the human genome contain? How are they expressed in different tissues? Are they evolutionarily conserved? Here, we present an analysis of 1,391 manually curated sequence-specific DNA-binding transcription factors, their functions, genomic organization and evolutionary conservation. Much remains to be explored, but this study provides a solid foundation for future investigations to elucidate regulatory mechanisms underlying diverse mammalian biological processes.

Title: Emerging roles of the EBF family of transcription factors in tumor suppression.
Abstract: Alterations in various developmental pathways are common themes in cancer. The early B-cell factors (EBF) are a family of four highly conserved DNA-binding transcription factors with an atypical zinc-finger and helix-loop-helix motif. They are involved in the differentiation and maturation of several cell lineages including B-progenitor lymphoblasts, neuronal precursors, and osteoblast progenitors. During B-cell development, EBF1 is required for the expression of Pax5, an essential factor for the production of antibody-secreting cells. Accumulating evidence indicates that genomic deletion of the EBF1 gene contributes to the pathogenesis, drug resistance, and relapse of B-progenitor acute lymphoblastic leukemia (ALL). Epigenetic silencing and genomic deletion of the EBF3 locus in chromosome 10q are very frequent in glioblastoma (GBM). Strikingly, the frequency of EBF3 loss in GBM is similar to that of the loss of Pten, a key suppressor of gliomagenesis. Cancer-specific somatic mutations were detected in EBF3 in GBM and in both EBF1 and EBF3 in pancreatic ductal adenocarcinoma. These missense mutations occur in the DNA-binding domain or the conserved IPT/TIG domain, suggesting that they might disrupt the functions of these two proteins. Functional studies revealed that EBF3 represses the expression of genes required for cell proliferation [e.g., cyclins and cyclin-dependent kinases (CDK)] and survival (e.g., Mcl-1 and Daxx) but activates those involved in cell cycle arrest (e.g., p21 and p27), leading to growth suppression and apoptosis. Therefore, EBFs represent new tumor suppressors whose inactivation blocks normal development and contributes to tumorigenesis of diverse types of human cancer.

Title: HPV status-independent association of alcohol and tobacco exposure or prior radiation therapy with promoter methylation of FUSSEL18, EBF3, IRX1, and SEPT9, but not SLC5A8, in head and neck squamous cell carcinomas.
Abstract: Head and neck squamous cell carcinoma (HNSCC) is an aggressive malignancy with more than half a million people being diagnosed with the disease annually. Within the last 2 decades, the human papillomavirus (HPV) has been found to be associated with this malignancy. More recently, HPV-infected HNSCC has been found to exhibit higher levels of global DNA methylation. In a recent study, we identified five tumor suppressive genes (IRX1, EBF3, SLC5A8, SEPT9, and FUSSEL18) as frequently methylated in HNSCC biopsies using a global methylation analysis via restriction landmark genomic scanning. In this study, we verify these genes as valid methylation markers in two separate sets of HNSCC specimens. By using the available clinical information linked to the patient specimens, we found a strong association between promoter methylation of FUSSEL18, IRX1, and EBF3 and prior radiation therapy (P &lt; 0.0001) irrespective of HPV status. Also, promoter methylation of FUSSEL18 and SEPTIN9 was found to correlate significantly with exposure to alcohol and tobacco (P = 0.021). Importantly, in this study, we preliminarily show a trend between HPV16 positivity and specific target gene hypermethylation of IRX1, EBF3, SLC5A8, and SEPT9. If replicated in a larger study, the HPV status may be a patient selection biomarker when determining the most efficacious treatment modality for these different subsets of patients (e.g., inclusion or exclusion of epigenetic therapies). Equally notable and independent of HPV status, hypermethylation of the promoters of a subset of these genes in recurrences especially in the setting of prior radiation or in the setting of alcohol and tobacco use might help guide adjunctive inclusion or exclusion or epigenetic therapy.

Title: Aberrant DNA methylation and tumor suppressive activity of the EBF3 gene in gastric carcinoma.
Abstract: The early B-cell factors (EBFs) are a group of four highly conserved DNA-binding transcription factors with an atypical zinc-finger and a helix-loop-helix domain. The EBF3 locus on chromosome 10q26.3 is epigenetically silenced or deleted in several types of cancers. In addition, EBF3 activates genes involved in cell cycle arrest and inhibits cell survival. However, inactivation of EBF3 gene expression was not fully studied in gastric carcinoma and the functions of EBF3 that underlie EBF3-regulated tumor suppression have not been identified. In our study, we found that inactivation of the EBF3 gene is frequently accompanied by promoter region hypermethylation in several gastric cancer cell lines and that the gene is reactivated by 5-aza-2'-deoxycytidine (5-aza-dc) and/or trichostatin A (TSA) in all ten gastric cancer cell lines. We performed functional analysis using small interfering RNA or expressional cDNA transfection in gastric cancer cell lines and demonstrate that EBF3 represses gastric cancer cell growth and migration, but activates cell cycle arrest and apoptosis. Promoter methylation of EBF3 was detected in 42/104 (40.4%) gastric cancer tissues but not in normal gastric tissues. Furthermore, promoter methylation of EBF3 was found to be significantly correlated with lymphatic invasion (p = 0.013) and poor survival (p = 0.038) in gastric carcinoma. These results suggest that EBF3 tumor suppressor is epigenetically silenced and that it serves as an independent prognostic marker in gastric carcinoma.

Title: Phylogenetic-based propagation of functional annotations within the Gene Ontology consortium.
Abstract: The goal of the Gene Ontology (GO) project is to provide a uniform way to describe the functions of gene products from organisms across all kingdoms of life and thereby enable analysis of genomic data. Protein annotations are either based on experiments or predicted from protein sequences. Since most sequences have not been experimentally characterized, most available annotations need to be based on predictions. To make as accurate inferences as possible, the GO Consortium's Reference Genome Project is using an explicit evolutionary framework to infer annotations of proteins from a broad set of genomes from experimental annotations in a semi-automated manner. Most components in the pipeline, such as selection of sequences, building multiple sequence alignments and phylogenetic trees, retrieving experimental annotations and depositing inferred annotations, are fully automated. However, the most crucial step in our pipeline relies on software-assisted curation by an expert biologist. This curation tool, Phylogenetic Annotation and INference Tool (PAINT) helps curators to infer annotations among members of a protein family. PAINT allows curators to make precise assertions as to when functions were gained and lost during evolution and record the evidence (e.g. experimentally supported GO annotations and phylogenetic information including orthology) for those assertions. In this article, we describe how we use PAINT to infer protein function in a phylogenetic context with emphasis on its strengths, limitations and guidelines. We also discuss specific examples showing how PAINT annotations compare with those generated by other highly used homology-based methods.

Title: A genome-wide association study of sleep habits and insomnia.
Abstract: Several aspects of sleep behavior such as timing, duration and quality have been demonstrated to be heritable. To identify common variants that influence sleep traits in the population, we conducted a genome-wide association study of six sleep phenotypes assessed by questionnaire in a sample of 2,323 individuals from the Australian Twin Registry. Genotyping was performed on the Illumina 317, 370, and 610K arrays and the SNPs in common between platforms were used to impute non-genotyped SNPs. We tested for association with more than 2,000,000 common polymorphisms across the genome. While no SNPs reached the genome-wide significance threshold, we identified a number of associations in plausible candidate genes. Most notably, a group of SNPs in the third intron of the CACNA1C gene ranked as most significant in the analysis of sleep latency (P = 1.3 × 10⁻⁶). We attempted to replicate this association in an independent sample from the Chronogen Consortium (n = 2,034), but found no evidence of association (P = 0.73). We have identified several other suggestive associations that await replication in an independent sample. We did not replicate the results from previous genome-wide analyses of self-reported sleep phenotypes after correction for multiple testing.

Title: Early B-cell factor 3 (EBF3) is a novel tumor suppressor gene with promoter hypermethylation in pediatric acute myeloid leukemia.
Abstract: Pediatric acute myeloid leukemia (AML) comprises up to 20% of all childhood leukemia. Recent research shows that aberrant DNA methylation patterning may play a role in leukemogenesis. The epigenetic silencing of the EBF3 locus is very frequent in glioblastoma. However, the expression profiles and molecular function of EBF3 in pediatric AML is still unclear.

Title: Proteomic analyses reveal distinct chromatin-associated and soluble transcription factor complexes.
Abstract: The current knowledge on how transcription factors (TFs), the ultimate targets and executors of cellular signalling pathways, are regulated by protein-protein interactions remains limited. Here, we performed proteomics analyses of soluble and chromatin-associated complexes of 56 TFs, including the targets of many signalling pathways involved in development and cancer, and 37 members of the Forkhead box (FOX) TF family. Using tandem affinity purification followed by mass spectrometry (TAP/MS), we performed 214 purifications and identified 2,156 high-confident protein-protein interactions. We found that most TFs form very distinct protein complexes on and off chromatin. Using this data set, we categorized the transcription-related or unrelated regulators for general or specific TFs. Our study offers a valuable resource of protein-protein interaction networks for a large number of TFs and underscores the general principle that TFs form distinct location-specific protein complexes that are associated with the different regulation and diverse functions of these TFs.

Title: Dependence of Human Colorectal Cells Lacking the FBW7 Tumor Suppressor on the Spindle Assembly Checkpoint.
Abstract: FBW7 (F-box and WD repeat domain containing 7), also known as FBXW7 or hCDC4, is a tumor suppressor gene mutated in a broad spectrum of cancer cell types. As a component of the SCF E3 ubiquitin ligase, FBW7 is responsible for specifically recognizing phosphorylated substrates, many important for tumor progression, and targeting them for ubiquitin-mediated degradation. Although the role of FBW7 as a tumor suppressor is well established, less well studied is how FBW7-mutated cancer cells might be targeted for selective killing. To explore this further, we undertook a genome-wide RNAi screen using WT and FBW7 knockout colorectal cell lines and identified the spindle assembly checkpoint (SAC) protein BUBR1, as a candidate synthetic lethal target. We show here that asynchronous FBW7 knockout cells have increased levels of mitotic APC/C substrates and are sensitive to knockdown of not just BUBR1 but BUB1 and MPS1, other known SAC components, suggesting a dependence of these cells on the mitotic checkpoint. Consistent with this dependence, knockdown of BUBR1 in cells lacking FBW7 results in significant cell aneuploidy and increases in p53 levels. The FBW7 substrate cyclin E was necessary for the genetic interaction with BUBR1. In contrast, the establishment of this dependence on the SAC requires the deregulation of multiple substrates of FBW7. Our work suggests that FBW7 knockout cells are vulnerable in their dependence on the mitotic checkpoint and that this may be a good potential target to exploit in FBW7-mutated cancer cells.

Title: An organelle-specific protein landscape identifies novel diseases and molecular mechanisms.
Abstract: Cellular organelles provide opportunities to relate biological mechanisms to disease. Here we use affinity proteomics, genetics and cell biology to interrogate cilia: poorly understood organelles, where defects cause genetic diseases. Two hundred and seventeen tagged human ciliary proteins create a final landscape of 1,319 proteins, 4,905 interactions and 52 complexes. Reverse tagging, repetition of purifications and statistical analyses, produce a high-resolution network that reveals organelle-specific interactions and complexes not apparent in larger studies, and links vesicle transport, the cytoskeleton, signalling and ubiquitination to ciliary signalling and proteostasis. We observe sub-complexes in exocyst and intraflagellar transport complexes, which we validate biochemically, and by probing structurally predicted, disruptive, genetic variants from ciliary disease patients. The landscape suggests other genetic diseases could be ciliary including 3M syndrome. We show that 3M genes are involved in ciliogenesis, and that patient fibroblasts lack cilia. Overall, this organelle-specific targeting strategy shows considerable promise for Systems Medicine.

Title: De Novo Mutations in EBF3 Cause a Neurodevelopmental Syndrome.
Abstract: Early B cell factor 3 (EBF3) is an atypical transcription factor that is thought to influence the laminar formation of the cerebral cortex. Here, we report that de novo mutations in EBF3 cause a complex neurodevelopmental syndrome. The mutations were identified in two large-scale sequencing projects: the UK Deciphering Developmental Disorders (DDD) study and the Canadian Clinical Assessment of the Utility of Sequencing and Evaluation as a Service (CAUSES) study. The core phenotype includes moderate to severe intellectual disability, and many individuals exhibit cerebellar ataxia, subtle facial dysmorphism, strabismus, and vesicoureteric reflux, suggesting that EBF3 has a widespread developmental role. Pathogenic de novo variants identified in EBF3 include multiple loss-of-function and missense mutations. Structural modeling suggested that the missense mutations affect DNA binding. Functional analysis of mutant proteins with missense substitutions revealed reduced transcriptional activities and abilities to form heterodimers with wild-type EBF3. We conclude that EBF3, a transcription factor previously unknown to be associated with human disease, is important for brain and other organ development and warrants further investigation.

Title: A Syndromic Neurodevelopmental Disorder Caused by De Novo Variants in EBF3.
Abstract: Early B cell factor 3 (EBF3) is a member of the highly evolutionarily conserved Collier/Olf/EBF (COE) family of transcription factors. Prior studies on invertebrate and vertebrate animals have shown that EBF3 homologs are essential for survival and that loss-of-function mutations are associated with a range of nervous system developmental defects, including perturbation of neuronal development and migration. Interestingly, aristaless-related homeobox (ARX), a homeobox-containing transcription factor critical for the regulation of nervous system development, transcriptionally represses EBF3 expression. However, human neurodevelopmental disorders related to EBF3 have not been reported. Here, we describe three individuals who are affected by global developmental delay, intellectual disability, and expressive speech disorder and carry de novo variants in EBF3. Associated features seen in these individuals include congenital hypotonia, structural CNS malformations, ataxia, and genitourinary abnormalities. The de novo variants affect a single conserved residue in a zinc finger motif crucial for DNA binding and are deleterious in a fly model. Our findings indicate that mutations in EBF3 cause a genetic neurodevelopmental syndrome and suggest that loss of EBF3 function might mediate a subset of neurologic phenotypes shared by ARX-related disorders, including intellectual disability, abnormal genitalia, and structural CNS malformations.

Title: Mutations in EBF3 Disturb Transcriptional Profiles and Cause Intellectual Disability, Ataxia, and Facial Dysmorphism.
Abstract: From a GeneMatcher-enabled international collaboration, we identified ten individuals affected by intellectual disability, speech delay, ataxia, and facial dysmorphism and carrying a deleterious EBF3 variant detected by whole-exome sequencing. One 9-bp duplication and one splice-site, five missense, and two nonsense variants in EBF3 were found; the mutations occurred de novo in eight individuals, and the missense variant c.625C&gt;T (p.Arg209Trp) was inherited by two affected siblings from their healthy mother, who is mosaic. EBF3 belongs to the early B cell factor family (also known as Olf, COE, or O/E) and is a transcription factor involved in neuronal differentiation and maturation. Structural assessment predicted that the five amino acid substitutions have damaging effects on DNA binding of EBF3. Transient expression of EBF3 mutant proteins in HEK293T cells revealed mislocalization of all but one mutant in the cytoplasm, as well as nuclear localization. By transactivation assays, all EBF3 mutants showed significantly reduced or no ability to activate transcription of the reporter gene CDKN1A, and in situ subcellular fractionation experiments demonstrated that EBF3 mutant proteins were less tightly associated with chromatin. Finally, in RNA-seq and ChIP-seq experiments, EBF3 acted as a transcriptional regulator, and mutant EBF3 had reduced genome-wide DNA binding and gene-regulatory activity. Our findings demonstrate that variants disrupting EBF3-mediated transcriptional regulation cause intellectual disability and developmental delay and are present in ∼0.1% of individuals with unexplained neurodevelopmental disorders.

Title: Genome-wide methylation sequencing of paired primary and metastatic cell lines identifies common DNA methylation changes and a role for EBF3 as a candidate epigenetic driver of melanoma metastasis.
Abstract: Epigenetic alterations are increasingly implicated in metastasis, whereas very few genetic mutations have been identified as authentic drivers of cancer metastasis. Yet, to date, few studies have identified metastasis-related epigenetic drivers, in part because a framework for identifying driver epigenetic changes in metastasis has not been established. Using reduced representation bisulfite sequencing (RRBS), we mapped genome-wide DNA methylation patterns in three cutaneous primary and metastatic melanoma cell line pairs to identify metastasis-related epigenetic drivers. Globally, metastatic melanoma cell lines were hypomethylated compared to the matched primary melanoma cell lines. Using whole genome RRBS we identified 75 shared (10 hyper- and 65 hypomethylated) differentially methylated fragments (DMFs), which were associated with 68 genes showing significant methylation differences. One gene, Early B Cell Factor 3 (EBF3), exhibited promoter hypermethylation in metastatic cell lines, and was validated with bisulfite sequencing and in two publicly available independent melanoma cohorts (n = 40 and 458 melanomas, respectively). We found that hypermethylation of the EBF3 promoter was associated with increased EBF3 mRNA levels in metastatic melanomas and subsequent inhibition of DNA methylation reduced EBF3 expression. RNAi-mediated knockdown of EBF3 mRNA levels decreased proliferation, migration and invasion in primary and metastatic melanoma cell lines. Overall, we have identified numerous epigenetic changes characterising metastatic melanoma cell lines, including EBF3-induced aggressive phenotypic behaviour with elevated EBF3 expression in metastatic melanoma, suggesting that EBF3 promoter hypermethylation may be a candidate epigenetic driver of metastasis.

Title: Architecture of the human interactome defines protein communities and disease networks.
Abstract: The physiology of a cell can be viewed as the product of thousands of proteins acting in concert to shape the cellular response. Coordination is achieved in part through networks of protein-protein interactions that assemble functionally related proteins into complexes, organelles, and signal transduction pathways. Understanding the architecture of the human proteome has the potential to inform cellular, structural, and evolutionary mechanisms and is critical to elucidating how genome variation contributes to disease. Here we present BioPlex 2.0 (Biophysical Interactions of ORFeome-derived complexes), which uses robust affinity purification-mass spectrometry methodology to elucidate protein interaction networks and co-complexes nucleated by more than 25% of protein-coding genes from the human genome, and constitutes, to our knowledge, the largest such network so far. With more than 56,000 candidate interactions, BioPlex 2.0 contains more than 29,000 previously unknown co-associations and provides functional insights into hundreds of poorly characterized proteins while enhancing network-based analyses of domain associations, subcellular localization, and co-complex formation. Unsupervised Markov clustering of interacting proteins identified more than 1,300 protein communities representing diverse cellular activities. Genes essential for cell fitness are enriched within 53 communities representing central cellular functions. Moreover, we identified 442 communities associated with more than 2,000 disease annotations, placing numerous candidate disease genes into a cellular framework. BioPlex 2.0 exceeds previous experimentally derived interaction networks in depth and breadth, and will be a valuable resource for exploring the biology of incompletely characterized proteins and for elucidating larger-scale patterns of proteome organization.

Title: De novo variants in <i>EBF3</i> are associated with hypotonia, developmental delay, intellectual disability, and autism.
Abstract: Using whole-exome sequencing, we identified seven unrelated individuals with global developmental delay, hypotonia, dysmorphic facial features, and an increased frequency of short stature, ataxia, and autism with de novo heterozygous frameshift, nonsense, splice, and missense variants in the <i>Early B-cell Transcription Factor Family Member 3</i> (<i>EBF3</i>) gene. EBF3 is a member of the collier/olfactory-1/early B-cell factor (COE) family of proteins, which are required for central nervous system (CNS) development. COE proteins are highly evolutionarily conserved and regulate neuronal specification, migration, axon guidance, and dendritogenesis during development and are essential for maintaining neuronal identity in adult neurons. Haploinsufficiency of <i>EBF3</i> may affect brain development and function, resulting in developmental delay, intellectual disability, and behavioral differences observed in individuals with a deleterious variant in <i>EBF3</i>.

Title: Rare Variant, Gene-Based Association Study of Hereditary Melanoma Using Whole-Exome Sequencing.
Abstract: Extraordinary progress has been made in our understanding of common variants in many diseases, including melanoma. Because the contribution of rare coding variants is not as well characterized, we performed an exome-wide, gene-based association study of familial cutaneous melanoma (CM) and ocular melanoma (OM).

Title: MiR-23b-3p induces the proliferation and metastasis of esophageal squamous cell carcinomas cells through the inhibition of EBF3.
Abstract: MicroRNAs (miRNAs), some small non-coding RNAs that regulate gene expression at the posttranscriptional level, are always aberrantly expressed in carcinomas. In this study, we found that miR-23b-3p was remarkably up-regulated in human esophageal squamous cell carcinoma cells and tissues. Moreover, miR-23b-3p could induce the proliferation, invasion, and metastasis in vitro. EBF3 was identified as the direct downstream target gene of miR-23b-3p and ectogenic EBF3 could strongly inhibit the proliferation, invasion, and metastasis in vitro. Furthermore, it was found that miR-23b-3p could regulate epithelial-to-mesenchymal transition progress by blocking EBF3. Therefore, it was concluded that miR-23b-3p targeted EBF3 to accelerate the proliferation, invasion, and metastasis in ESCC.

Title: Associations of Pulmonary Fibrosis with Peripheral Blood Th1/Th2 Cell Imbalance and EBF3 Gene Methylation in Uygur Pigeon Breeder's Lung Patients.
Abstract: Background/Aims Pigeon breeder's lung (PBL) results from Th1/Th2 cell imbalance. B cells inhibit the immune activity of Th1, and EBF3 is a key B cell factor. This study explored the relationship between EBF3 and Th1/Th2 imbalance in chronic PBL cases complicated with pulmonary fibrosis (PF). Methods Twenty Uygur PBL+PF patients, 20 pigeon breeders without PBL or PF, and 20 healthy individuals without pigeon breeding history constituted the patient I, negative control, and normal control groups, respectively. Peripheral blood specimens and case backgrounds were collected between June 2016 and March 2017. EBF3 gene methylation was analyzed by matrix assisted laser desorption ionization-time of flight mass spectrometry. To compare different mechanisms of PF progression in PBL, samples from 20 Uygur PBL patients without PF (at acute and sub-acute stages) were collected between October 2017 and February 2018, constituting the patient II group. EBF3 mRNA expression was evaluated by real-time polymerase chain reaction. IFN-γ, IL-4 and IL-10 expression and Th1/Th2 imbalance in PBL were evaluated by enzyme-linked immunosorbent assay and flow cytometry. Results CpG-2 and general methylation rates in the patient I group were lower than those in the control groups (P˂0.017). The level of EBF3 mRNA expression in the patient I group was significantly higher than that in any other group. Compared with the control groups, the patient I group showed a significantly higher level of IL-4, whereas the patient II group showed a significantly lower level. IL-10 was also expressed more highly in the patient I group than in any other group (P&lt; 0.01). Flow cytometry showed INF-γ dominance (Th1 cytokine) in PBL at the acute/sub-acute stage and IL-4 dominance (Th2 cytokine) at the chronic stage after PF occurred. The general methylation rate was negatively correlated with the mRNA level, with the latter being positively correlated with the IL-10 level and number of pigeons bred in the past 3 months. IL-4 expression was negatively correlated with INF-γ but positively correlated with PF area and duration of pigeon breeding history. Conclusions After PF occurs in chronic PBL, the inflammation type changes from Th1 dominance to Th2 dominance. During PBL development, IL-10 increases before IL-4 does, which may be associated with EBF3 hypomethylation and the involvement of B lymphocytes.
```

get clinical significance assertions for rsID rs4987049 using `rsid_clinsig.sh` 

```
$ bash rsid_clinsig.sh rs4987049

>>> rs4987049
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000257043.2
	Breast-ovarian cancer, familial 2
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000479392.1
	not provided
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000045914.12
	Hereditary breast and ovarian cancer syndrome
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000077479.8
	Breast-ovarian cancer, familial 2
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000131045.6
	Hereditary cancer-predisposing syndrome
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000221171.4
	not provided
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000768644.1
	Breast and/or ovarian cancer
	Pathogenic
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000045915.4
	not specified
	Benign
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000112823.2
	Breast-ovarian cancer, familial 2
	Benign
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000163121.3
	Hereditary cancer-predisposing syndrome
	Benign-Likely-Benign
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000195310.5
	Hereditary breast and ovarian cancer syndrome
	Benign-Likely-Benign
	
	https://www.ncbi.nlm.nih.gov/clinvar/RCV000590428.1
	not provided
	Likely-Benign

```

## Considerations
scripts are parsing web pages (probably a bad idea), they could break if source page code changes
